2022-04-22 02:05:34,877:DEBUG::
2022-04-22 02:05:34,878:DEBUG::====================================================================================================
2022-04-22 02:05:34,878:INFO::Server Utils Section :: Logging Active
2022-04-22 02:05:34,879:DEBUG::
2022-04-22 02:05:41,432:DEBUG::
2022-04-22 02:05:41,432:DEBUG::====================================================================================================
2022-04-22 02:05:41,433:INFO::Server Utils Section :: Logging Active
2022-04-22 02:05:41,433:DEBUG::
2022-04-22 02:05:50,922:DEBUG::
2022-04-22 02:05:50,922:DEBUG::====================================================================================================
2022-04-22 02:05:50,923:INFO::Server Utils Section :: Logging Active
2022-04-22 02:05:50,923:DEBUG::
2022-04-22 02:05:55,082:DEBUG::
2022-04-22 02:05:55,083:DEBUG::====================================================================================================
2022-04-22 02:05:55,083:INFO::Server Utils Section :: Logging Active
2022-04-22 02:05:55,083:DEBUG::
2022-04-22 02:06:06,492:DEBUG::
2022-04-22 02:06:06,492:DEBUG::====================================================================================================
2022-04-22 02:06:06,492:INFO::Server Utils Section :: Logging Active
2022-04-22 02:06:06,493:DEBUG::
2022-04-22 02:09:59,177:DEBUG::AddNewProductError: Failed to Add New Product,[Errno 2] No such file or directory: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\app/static/images/products/admin_product.jpg'
2022-04-22 02:11:06,656:DEBUG::
2022-04-22 02:11:06,657:DEBUG::====================================================================================================
2022-04-22 02:11:06,657:INFO::Server Utils Section :: Logging Active
2022-04-22 02:11:06,657:DEBUG::
2022-04-22 02:11:23,416:DEBUG::AddNewProductError: Failed to Add New Product,[Errno 2] No such file or directory: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/admin_product.jpg'
2022-04-22 02:13:11,590:DEBUG::AddNewProductError: Failed to Add New Product,local variable 'checkID' referenced before assignment
2022-04-22 02:14:44,943:DEBUG::AddNewProductError: Failed to Add New Product,cannot write mode RGBA as JPEG
2022-04-22 02:19:37,000:DEBUG::
2022-04-22 02:19:37,000:DEBUG::====================================================================================================
2022-04-22 02:19:37,000:INFO::Server Utils Section :: Logging Active
2022-04-22 02:19:37,001:DEBUG::
2022-04-22 02:19:42,244:DEBUG::AddNewProductError: Failed to Add New Product,'bytes' object has no attribute 'format'
2022-04-22 02:21:02,897:DEBUG::
2022-04-22 02:21:02,897:DEBUG::====================================================================================================
2022-04-22 02:21:02,897:INFO::Server Utils Section :: Logging Active
2022-04-22 02:21:02,898:DEBUG::
2022-04-22 02:21:07,572:DEBUG::
2022-04-22 02:21:07,572:DEBUG::====================================================================================================
2022-04-22 02:21:07,572:INFO::Server Utils Section :: Logging Active
2022-04-22 02:21:07,573:DEBUG::
2022-04-22 02:21:14,730:DEBUG::AddNewProductError: Failed to Add New Product,'bytes' object has no attribute 'format'
2022-04-22 02:21:27,543:DEBUG::
2022-04-22 02:21:27,543:DEBUG::====================================================================================================
2022-04-22 02:21:27,544:INFO::Server Utils Section :: Logging Active
2022-04-22 02:21:27,544:DEBUG::
2022-04-22 02:21:30,279:DEBUG::AddNewProductError: Failed to Add New Product,local variable 'checkID' referenced before assignment
2022-04-22 02:22:47,925:DEBUG::
2022-04-22 02:22:47,925:DEBUG::====================================================================================================
2022-04-22 02:22:47,925:INFO::Server Utils Section :: Logging Active
2022-04-22 02:22:47,926:DEBUG::
2022-04-22 02:22:59,037:DEBUG::AddNewProductError: Failed to Add New Product,cannot write mode RGBA as JPEG
2022-04-22 02:29:18,834:DEBUG::
2022-04-22 02:29:18,834:DEBUG::====================================================================================================
2022-04-22 02:29:18,835:INFO::Server Utils Section :: Logging Active
2022-04-22 02:29:18,835:DEBUG::
2022-04-22 02:30:21,039:DEBUG::
2022-04-22 02:30:21,039:DEBUG::====================================================================================================
2022-04-22 02:30:21,039:INFO::Server Utils Section :: Logging Active
2022-04-22 02:30:21,040:DEBUG::
2022-04-22 02:33:32,963:DEBUG::
2022-04-22 02:33:32,963:DEBUG::====================================================================================================
2022-04-22 02:33:32,963:INFO::Server Utils Section :: Logging Active
2022-04-22 02:33:32,964:DEBUG::
2022-04-22 02:34:52,844:DEBUG::
2022-04-22 02:34:52,845:DEBUG::====================================================================================================
2022-04-22 02:34:52,845:INFO::Server Utils Section :: Logging Active
2022-04-22 02:34:52,846:DEBUG::
2022-04-22 02:37:05,039:DEBUG::
2022-04-22 02:37:05,039:DEBUG::====================================================================================================
2022-04-22 02:37:05,039:INFO::Server Utils Section :: Logging Active
2022-04-22 02:37:05,040:DEBUG::
2022-04-22 02:38:00,238:DEBUG::
2022-04-22 02:38:00,238:DEBUG::====================================================================================================
2022-04-22 02:38:00,238:INFO::Server Utils Section :: Logging Active
2022-04-22 02:38:00,239:DEBUG::
2022-04-22 02:39:10,389:DEBUG::
2022-04-22 02:39:10,390:DEBUG::====================================================================================================
2022-04-22 02:39:10,390:INFO::Server Utils Section :: Logging Active
2022-04-22 02:39:10,391:DEBUG::
2022-04-22 02:40:41,280:DEBUG::
2022-04-22 02:40:41,280:DEBUG::====================================================================================================
2022-04-22 02:40:41,280:INFO::Server Utils Section :: Logging Active
2022-04-22 02:40:41,281:DEBUG::
2022-04-22 02:43:47,423:DEBUG::
2022-04-22 02:43:47,423:DEBUG::====================================================================================================
2022-04-22 02:43:47,423:INFO::Server Utils Section :: Logging Active
2022-04-22 02:43:47,424:DEBUG::
2022-04-22 02:46:53,131:DEBUG::
2022-04-22 02:46:53,132:DEBUG::====================================================================================================
2022-04-22 02:46:53,132:INFO::Server Utils Section :: Logging Active
2022-04-22 02:46:53,133:DEBUG::
2022-04-22 03:19:34,741:DEBUG::
2022-04-22 03:19:34,741:DEBUG::====================================================================================================
2022-04-22 03:19:34,742:INFO::Server Utils Section :: Logging Active
2022-04-22 03:19:34,742:DEBUG::
2022-04-22 10:45:01,493:DEBUG::
2022-04-22 10:45:01,493:DEBUG::====================================================================================================
2022-04-22 10:45:01,494:INFO::Server Utils Section :: Logging Active
2022-04-22 10:45:01,494:DEBUG::
2022-04-22 10:45:48,897:DEBUG::
2022-04-22 10:45:48,899:DEBUG::====================================================================================================
2022-04-22 10:45:48,899:INFO::Server Utils Section :: Logging Active
2022-04-22 10:45:48,899:DEBUG::
2022-04-22 10:45:59,726:DEBUG::
2022-04-22 10:45:59,727:DEBUG::====================================================================================================
2022-04-22 10:45:59,727:INFO::Server Utils Section :: Logging Active
2022-04-22 10:45:59,727:DEBUG::
2022-04-22 10:46:02,997:DEBUG::
2022-04-22 10:46:02,997:DEBUG::====================================================================================================
2022-04-22 10:46:02,998:INFO::Server Utils Section :: Logging Active
2022-04-22 10:46:02,999:DEBUG::
2022-04-22 10:46:23,017:DEBUG::
2022-04-22 10:46:23,017:DEBUG::====================================================================================================
2022-04-22 10:46:23,018:INFO::Server Utils Section :: Logging Active
2022-04-22 10:46:23,018:DEBUG::
2022-04-22 10:52:55,652:DEBUG::
2022-04-22 10:52:55,652:DEBUG::====================================================================================================
2022-04-22 10:52:55,653:INFO::Server Utils Section :: Logging Active
2022-04-22 10:52:55,653:DEBUG::
2022-04-22 11:06:21,410:DEBUG::
2022-04-22 11:06:21,411:DEBUG::====================================================================================================
2022-04-22 11:06:21,411:INFO::Server Utils Section :: Logging Active
2022-04-22 11:06:21,411:DEBUG::
2022-04-22 11:15:49,484:DEBUG::
2022-04-22 11:15:49,484:DEBUG::====================================================================================================
2022-04-22 11:15:49,485:INFO::Server Utils Section :: Logging Active
2022-04-22 11:15:49,485:DEBUG::
2022-04-22 11:16:19,407:DEBUG::
2022-04-22 11:16:19,407:DEBUG::====================================================================================================
2022-04-22 11:16:19,408:INFO::Server Utils Section :: Logging Active
2022-04-22 11:16:19,408:DEBUG::
2022-04-22 11:43:47,811:DEBUG::
2022-04-22 11:43:47,811:DEBUG::====================================================================================================
2022-04-22 11:43:47,811:INFO::Server Utils Section :: Logging Active
2022-04-22 11:43:47,811:DEBUG::
2022-04-22 11:43:52,215:DEBUG::
2022-04-22 11:43:52,215:DEBUG::====================================================================================================
2022-04-22 11:43:52,215:INFO::Server Utils Section :: Logging Active
2022-04-22 11:43:52,216:DEBUG::
2022-04-22 11:53:14,657:DEBUG::
2022-04-22 11:53:14,658:DEBUG::====================================================================================================
2022-04-22 11:53:14,658:INFO::Server Utils Section :: Logging Active
2022-04-22 11:53:14,658:DEBUG::
2022-04-22 11:53:18,271:DEBUG::
2022-04-22 11:53:18,271:DEBUG::====================================================================================================
2022-04-22 11:53:18,272:INFO::Server Utils Section :: Logging Active
2022-04-22 11:53:18,272:DEBUG::
2022-04-22 12:01:20,617:DEBUG::
2022-04-22 12:01:20,618:DEBUG::====================================================================================================
2022-04-22 12:01:20,619:INFO::Server Utils Section :: Logging Active
2022-04-22 12:01:20,620:DEBUG::
2022-04-22 12:26:07,206:DEBUG::
2022-04-22 12:26:07,206:DEBUG::====================================================================================================
2022-04-22 12:26:07,206:INFO::Server Utils Section :: Logging Active
2022-04-22 12:26:07,207:DEBUG::
2022-04-22 15:16:03,281:DEBUG::
2022-04-22 15:16:03,281:DEBUG::====================================================================================================
2022-04-22 15:16:03,282:INFO::Server Utils Section :: Logging Active
2022-04-22 15:16:03,282:DEBUG::
2022-04-22 15:18:23,247:DEBUG::
2022-04-22 15:18:23,248:DEBUG::====================================================================================================
2022-04-22 15:18:23,248:INFO::Server Utils Section :: Logging Active
2022-04-22 15:18:23,248:DEBUG::
2022-04-22 15:34:34,590:DEBUG::
2022-04-22 15:34:34,591:DEBUG::====================================================================================================
2022-04-22 15:34:34,592:INFO::Server Utils Section :: Logging Active
2022-04-22 15:34:34,593:DEBUG::
2022-04-22 15:39:01,728:DEBUG::
2022-04-22 15:39:01,729:DEBUG::====================================================================================================
2022-04-22 15:39:01,729:INFO::Server Utils Section :: Logging Active
2022-04-22 15:39:01,729:DEBUG::
2022-04-22 15:44:12,740:DEBUG::
2022-04-22 15:44:12,740:DEBUG::====================================================================================================
2022-04-22 15:44:12,741:INFO::Server Utils Section :: Logging Active
2022-04-22 15:44:12,741:DEBUG::
2022-04-22 15:46:38,015:DEBUG::
2022-04-22 15:46:38,019:DEBUG::====================================================================================================
2022-04-22 15:46:38,019:INFO::Server Utils Section :: Logging Active
2022-04-22 15:46:38,020:DEBUG::
2022-04-22 15:46:56,813:DEBUG::EditProductError: Failed to Edit Product,'Product' object has no attribute 'discount'
2022-04-22 15:49:42,273:DEBUG::
2022-04-22 15:49:42,274:DEBUG::====================================================================================================
2022-04-22 15:49:42,274:INFO::Server Utils Section :: Logging Active
2022-04-22 15:49:42,275:DEBUG::
2022-04-22 15:52:57,205:DEBUG::EditProductError: Failed to Edit Product,'Product' object has no attribute 'product_image'
2022-04-22 15:54:45,561:DEBUG::EditProductError: Failed to Edit Product,'Product' object has no attribute 'product_image'
2022-04-22 15:56:35,832:DEBUG::
2022-04-22 15:56:35,833:DEBUG::====================================================================================================
2022-04-22 15:56:35,834:INFO::Server Utils Section :: Logging Active
2022-04-22 15:56:35,834:DEBUG::
2022-04-22 16:02:43,784:DEBUG::
2022-04-22 16:02:43,785:DEBUG::====================================================================================================
2022-04-22 16:02:43,785:INFO::Server Utils Section :: Logging Active
2022-04-22 16:02:43,785:DEBUG::
2022-04-22 16:02:55,346:DEBUG::EditProductError: Failed to Edit Product,[WinError 2] The system cannot find the file specified: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/54e30ef765e3d3fed9324c1393284659_agroai2.png'
2022-04-22 16:05:50,847:DEBUG::EditProductError: Failed to Edit Product,[WinError 2] The system cannot find the file specified: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/54e30ef765e3d3fed9324c1393284659_agroai2.png'
2022-04-22 16:07:09,618:DEBUG::
2022-04-22 16:07:09,618:DEBUG::====================================================================================================
2022-04-22 16:07:09,619:INFO::Server Utils Section :: Logging Active
2022-04-22 16:07:09,619:DEBUG::
2022-04-22 16:07:20,072:DEBUG::EditProductError: Failed to Edit Product,[WinError 2] The system cannot find the file specified: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/54e30ef765e3d3fed9324c1393284659_agroai2.png'
2022-04-22 16:09:24,437:DEBUG::
2022-04-22 16:09:24,437:DEBUG::====================================================================================================
2022-04-22 16:09:24,438:INFO::Server Utils Section :: Logging Active
2022-04-22 16:09:24,438:DEBUG::
2022-04-22 16:09:33,702:DEBUG::EditProductError: Failed to Edit Product,[WinError 2] The system cannot find the file specified: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/54e30ef765e3d3fed9324c1393284659_agroai2.png'
2022-04-22 16:10:12,303:DEBUG::
2022-04-22 16:10:12,303:DEBUG::====================================================================================================
2022-04-22 16:10:12,304:INFO::Server Utils Section :: Logging Active
2022-04-22 16:10:12,304:DEBUG::
2022-04-22 16:10:25,908:DEBUG::EditProductError: Failed to Edit Product,[WinError 2] The system cannot find the file specified: 'C:\\Users\\Adefolahan\\WEB_APPS\\Doxael\\Api_Server\\app\\static/images/products/54e30ef765e3d3fed9324c1393284659_agroai2.png'
2022-04-22 16:18:06,958:DEBUG::
2022-04-22 16:18:06,958:DEBUG::====================================================================================================
2022-04-22 16:18:06,958:INFO::Server Utils Section :: Logging Active
2022-04-22 16:18:06,959:DEBUG::
2022-04-22 16:18:24,264:DEBUG::
2022-04-22 16:18:24,264:DEBUG::====================================================================================================
2022-04-22 16:18:24,265:INFO::Server Utils Section :: Logging Active
2022-04-22 16:18:24,266:DEBUG::
2022-04-22 21:22:03,943:DEBUG::
2022-04-22 21:22:03,946:DEBUG::====================================================================================================
2022-04-22 21:22:03,947:INFO::Server Utils Section :: Logging Active
2022-04-22 21:22:03,947:DEBUG::
2022-04-22 21:32:37,232:DEBUG::
2022-04-22 21:32:37,232:DEBUG::====================================================================================================
2022-04-22 21:32:37,232:INFO::Server Utils Section :: Logging Active
2022-04-22 21:32:37,233:DEBUG::
2022-04-22 21:33:07,756:DEBUG::
2022-04-22 21:33:07,756:DEBUG::====================================================================================================
2022-04-22 21:33:07,756:INFO::Server Utils Section :: Logging Active
2022-04-22 21:33:07,757:DEBUG::
2022-04-22 22:06:19,198:DEBUG::
2022-04-22 22:06:19,198:DEBUG::====================================================================================================
2022-04-22 22:06:19,199:INFO::Server Utils Section :: Logging Active
2022-04-22 22:06:19,199:DEBUG::
2022-04-22 22:07:34,975:DEBUG::
2022-04-22 22:07:34,975:DEBUG::====================================================================================================
2022-04-22 22:07:34,976:INFO::Server Utils Section :: Logging Active
2022-04-22 22:07:34,976:DEBUG::
2022-04-22 22:08:23,734:DEBUG::
2022-04-22 22:08:23,734:DEBUG::====================================================================================================
2022-04-22 22:08:23,734:INFO::Server Utils Section :: Logging Active
2022-04-22 22:08:23,735:DEBUG::
2022-04-22 22:09:05,382:DEBUG::
2022-04-22 22:09:05,382:DEBUG::====================================================================================================
2022-04-22 22:09:05,383:INFO::Server Utils Section :: Logging Active
2022-04-22 22:09:05,383:DEBUG::
2022-04-22 22:10:24,502:DEBUG::
2022-04-22 22:10:24,502:DEBUG::====================================================================================================
2022-04-22 22:10:24,502:INFO::Server Utils Section :: Logging Active
2022-04-22 22:10:24,502:DEBUG::
2022-04-22 22:14:54,223:DEBUG::
2022-04-22 22:14:54,223:DEBUG::====================================================================================================
2022-04-22 22:14:54,225:INFO::Server Utils Section :: Logging Active
2022-04-22 22:14:54,225:DEBUG::
2022-04-22 22:26:09,982:DEBUG::
2022-04-22 22:26:09,983:DEBUG::====================================================================================================
2022-04-22 22:26:09,983:INFO::Server Utils Section :: Logging Active
2022-04-22 22:26:09,983:DEBUG::
2022-04-22 22:45:09,169:DEBUG::
2022-04-22 22:45:09,170:DEBUG::====================================================================================================
2022-04-22 22:45:09,170:INFO::Server Utils Section :: Logging Active
2022-04-22 22:45:09,171:DEBUG::
2022-04-22 22:45:15,015:DEBUG::MakeFeaturedProductError: Failed to Make Product Featured,(MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'products.product_is_featured' in 'field list'")
[SQL: SELECT products.id AS products_id, products.product_id AS products_product_id, products.product_name AS products_product_name, products.product_description AS products_product_description, products.product_price AS products_product_price, products.product_image_name AS products_product_image_name, products.product_image_filepath AS products_product_image_filepath, products.product_discount AS products_product_discount, products.product_owner AS products_product_owner, products.product_is_available AS products_product_is_available, products.product_is_featured AS products_product_is_featured 
FROM products 
WHERE products.product_id = %s 
 LIMIT %s]
[parameters: ('17d32c0c9bfac93d2df8ce4edb79b340', 1)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
2022-04-22 22:46:02,460:DEBUG::
2022-04-22 22:46:02,460:DEBUG::====================================================================================================
2022-04-22 22:46:02,460:INFO::Server Utils Section :: Logging Active
2022-04-22 22:46:02,461:DEBUG::
2022-04-22 22:46:30,166:DEBUG::
2022-04-22 22:46:30,166:DEBUG::====================================================================================================
2022-04-22 22:46:30,167:INFO::Server Utils Section :: Logging Active
2022-04-22 22:46:30,167:DEBUG::
2022-04-22 22:47:28,824:DEBUG::
2022-04-22 22:47:28,824:DEBUG::====================================================================================================
2022-04-22 22:47:28,825:INFO::Server Utils Section :: Logging Active
2022-04-22 22:47:28,825:DEBUG::
2022-04-22 22:58:58,691:DEBUG::
2022-04-22 22:58:58,691:DEBUG::====================================================================================================
2022-04-22 22:58:58,692:INFO::Server Utils Section :: Logging Active
2022-04-22 22:58:58,692:DEBUG::
2022-04-23 00:02:23,521:DEBUG::
2022-04-23 00:02:23,521:DEBUG::====================================================================================================
2022-04-23 00:02:23,522:INFO::Server Utils Section :: Logging Active
2022-04-23 00:02:23,522:DEBUG::
2022-04-23 00:02:34,514:DEBUG::
2022-04-23 00:02:34,515:DEBUG::====================================================================================================
2022-04-23 00:02:34,515:INFO::Server Utils Section :: Logging Active
2022-04-23 00:02:34,515:DEBUG::
2022-04-23 00:02:43,241:DEBUG::
2022-04-23 00:02:43,242:DEBUG::====================================================================================================
2022-04-23 00:02:43,242:INFO::Server Utils Section :: Logging Active
2022-04-23 00:02:43,242:DEBUG::
2022-04-23 00:15:06,713:DEBUG::
2022-04-23 00:15:06,713:DEBUG::====================================================================================================
2022-04-23 00:15:06,714:INFO::Server Utils Section :: Logging Active
2022-04-23 00:15:06,714:DEBUG::
2022-04-23 00:15:27,277:DEBUG::
2022-04-23 00:15:27,277:DEBUG::====================================================================================================
2022-04-23 00:15:27,278:INFO::Server Utils Section :: Logging Active
2022-04-23 00:15:27,278:DEBUG::
2022-04-23 00:17:39,797:DEBUG::ToggleEnableCustomerError: Failed to Toggle Enable Vendor,Entity namespace for "customers" has no property "cutomer_id"
2022-04-23 00:18:20,004:DEBUG::
2022-04-23 00:18:20,005:DEBUG::====================================================================================================
2022-04-23 00:18:20,005:INFO::Server Utils Section :: Logging Active
2022-04-23 00:18:20,005:DEBUG::
2022-04-23 16:46:37,835:DEBUG::
2022-04-23 16:46:37,835:DEBUG::====================================================================================================
2022-04-23 16:46:37,835:INFO::Server Utils Section :: Logging Active
2022-04-23 16:46:37,836:DEBUG::
2022-04-26 19:53:51,940:DEBUG::
2022-04-26 19:53:51,940:DEBUG::====================================================================================================
2022-04-26 19:53:51,940:INFO::Server Utils Section :: Logging Active
2022-04-26 19:53:51,941:DEBUG::
2022-04-26 20:03:56,604:DEBUG::
2022-04-26 20:03:56,604:DEBUG::====================================================================================================
2022-04-26 20:03:56,604:INFO::Server Utils Section :: Logging Active
2022-04-26 20:03:56,605:DEBUG::
2022-04-26 20:05:01,339:DEBUG::ResetPasswordError: Failed to Reset Password,(MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'admin.admin_reset_pin' in 'field list'")
[SQL: SELECT `admin`.id AS admin_id_1, `admin`.admin_id AS admin_admin_id, `admin`.admin_name AS admin_admin_name, `admin`.admin_email AS admin_admin_email, `admin`.password AS admin_password, `admin`.admin_reset_pin AS admin_admin_reset_pin, `admin`.admin_push_notification_token AS admin_admin_push_notification_token 
FROM `admin` 
WHERE `admin`.admin_reset_pin = %s]
[parameters: ('644909',)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
2022-04-26 20:05:29,759:DEBUG::ResetPasswordError: Failed to Reset Password,(MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'admin.admin_reset_pin' in 'field list'")
[SQL: SELECT `admin`.id AS admin_id_1, `admin`.admin_id AS admin_admin_id, `admin`.admin_name AS admin_admin_name, `admin`.admin_email AS admin_admin_email, `admin`.password AS admin_password, `admin`.admin_reset_pin AS admin_admin_reset_pin, `admin`.admin_push_notification_token AS admin_admin_push_notification_token 
FROM `admin` 
WHERE `admin`.admin_reset_pin = %s]
[parameters: ('057634',)]
(Background on this error at: https://sqlalche.me/e/14/e3q8)
2022-04-26 20:05:53,764:DEBUG::
2022-04-26 20:05:53,764:DEBUG::====================================================================================================
2022-04-26 20:05:53,764:INFO::Server Utils Section :: Logging Active
2022-04-26 20:05:53,764:DEBUG::
2022-04-26 20:06:04,492:DEBUG::
2022-04-26 20:06:04,492:DEBUG::====================================================================================================
2022-04-26 20:06:04,493:INFO::Server Utils Section :: Logging Active
2022-04-26 20:06:04,493:DEBUG::
2022-04-26 20:06:19,139:DEBUG::
2022-04-26 20:06:19,139:DEBUG::====================================================================================================
2022-04-26 20:06:19,140:INFO::Server Utils Section :: Logging Active
2022-04-26 20:06:19,140:DEBUG::
2022-04-26 20:06:26,836:DEBUG::ResetPasswordError: Failed to Reset Password,'NoneType' object has no attribute 'admin_reset_pin'
2022-04-26 20:13:43,420:DEBUG::
2022-04-26 20:13:43,421:DEBUG::====================================================================================================
2022-04-26 20:13:43,421:INFO::Server Utils Section :: Logging Active
2022-04-26 20:13:43,421:DEBUG::
2022-04-26 20:16:15,107:DEBUG::
2022-04-26 20:16:15,108:DEBUG::====================================================================================================
2022-04-26 20:16:15,108:INFO::Server Utils Section :: Logging Active
2022-04-26 20:16:15,109:DEBUG::
2022-04-27 00:13:00,581:DEBUG::
2022-04-27 00:13:00,581:DEBUG::====================================================================================================
2022-04-27 00:13:00,581:INFO::Server Utils Section :: Logging Active
2022-04-27 00:13:00,582:DEBUG::
2022-04-27 00:13:07,474:DEBUG::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001D16FEBDD50>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 05:31:09,819:DEBUG::
2022-04-27 05:31:09,819:DEBUG::====================================================================================================
2022-04-27 05:31:09,820:INFO::Server Utils Section :: Logging Active
2022-04-27 05:31:09,820:DEBUG::
2022-04-27 06:00:06,088:DEBUG::
2022-04-27 06:00:06,088:DEBUG::====================================================================================================
2022-04-27 06:00:06,089:INFO::Server Utils Section :: Logging Active
2022-04-27 06:00:06,089:DEBUG::
2022-04-27 06:02:36,609:DEBUG::
2022-04-27 06:02:36,609:DEBUG::====================================================================================================
2022-04-27 06:02:36,610:INFO::Server Utils Section :: Logging Active
2022-04-27 06:02:36,610:DEBUG::
2022-04-27 06:02:41,031:DEBUG::ResetPasswordError: Failed to Reset Password,Object of type datetime is not JSON serializable
2022-04-27 06:04:02,825:DEBUG::
2022-04-27 06:04:02,826:DEBUG::====================================================================================================
2022-04-27 06:04:02,826:INFO::Server Utils Section :: Logging Active
2022-04-27 06:04:02,827:DEBUG::
2022-04-27 06:04:06,172:DEBUG::ResetPasswordError: Failed to Reset Password,Object of type datetime is not JSON serializable
2022-04-27 06:04:53,901:DEBUG::
2022-04-27 06:04:53,901:DEBUG::====================================================================================================
2022-04-27 06:04:53,902:INFO::Server Utils Section :: Logging Active
2022-04-27 06:04:53,902:DEBUG::
2022-04-27 06:05:01,354:ERROR::ResetPasswordError: Failed to Reset Password,Object of type datetime is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1142, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1094, in Send_Reset_Mail
    token = jwt.encode({'reset_pin':reset_pin,'email':email,'expiration': datetime.datetime.utcnow() + datetime.timedelta(minutes=13)},os.environ.get('SECRETE_KEY'))
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 59, in encode
    json_payload = json.dumps(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type datetime is not JSON serializable
2022-04-27 06:06:52,316:DEBUG::
2022-04-27 06:06:52,316:DEBUG::====================================================================================================
2022-04-27 06:06:52,317:INFO::Server Utils Section :: Logging Active
2022-04-27 06:06:52,317:DEBUG::
2022-04-27 06:07:04,413:ERROR::ResetPasswordError: Failed to Reset Password,Object of type datetime is not JSON serializable
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1144, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1096, in Send_Reset_Mail
    token = jwt.encode({'reset_pin':reset_pin,'email':email,'expiration': expiry_time},os.environ.get('SECRETE_KEY'))
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 59, in encode
    json_payload = json.dumps(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\json\encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type datetime is not JSON serializable
2022-04-27 06:07:52,523:DEBUG::
2022-04-27 06:07:52,523:DEBUG::====================================================================================================
2022-04-27 06:07:52,523:INFO::Server Utils Section :: Logging Active
2022-04-27 06:07:52,524:DEBUG::
2022-04-27 06:09:34,244:DEBUG::
2022-04-27 06:09:34,244:DEBUG::====================================================================================================
2022-04-27 06:09:34,244:INFO::Server Utils Section :: Logging Active
2022-04-27 06:09:34,245:DEBUG::
2022-04-27 06:09:43,303:ERROR::ResetPasswordError: Failed to Reset Password,'str' object has no attribute 'decode'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1144, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1096, in Send_Reset_Mail
    token = jwt.encode({'reset_pin':reset_pin,'email':email,'expiration': f"{expiry_time}"},os.environ.get('SECRETE_KEY')).decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
2022-04-27 06:10:52,963:DEBUG::
2022-04-27 06:10:52,964:DEBUG::====================================================================================================
2022-04-27 06:10:52,964:INFO::Server Utils Section :: Logging Active
2022-04-27 06:10:52,964:DEBUG::
2022-04-27 06:12:33,988:DEBUG::
2022-04-27 06:12:33,989:DEBUG::====================================================================================================
2022-04-27 06:12:33,989:INFO::Server Utils Section :: Logging Active
2022-04-27 06:12:33,989:DEBUG::
2022-04-27 06:34:27,516:DEBUG::
2022-04-27 06:34:27,517:DEBUG::====================================================================================================
2022-04-27 06:34:27,517:INFO::Server Utils Section :: Logging Active
2022-04-27 06:34:27,518:DEBUG::
2022-04-27 06:38:21,978:ERROR::UpdatePasswordError: Failed to Update Password,It is required that you pass in a value for the "algorithms" argument when calling decode().
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1153, in Update_Password
    data = jwt.decode(token,os.environ.get('SECRETE_KEY'))
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 119, in decode
    decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 86, in decode_complete
    raise DecodeError(
jwt.exceptions.DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-04-27 06:41:12,955:DEBUG::
2022-04-27 06:41:12,955:DEBUG::====================================================================================================
2022-04-27 06:41:12,956:INFO::Server Utils Section :: Logging Active
2022-04-27 06:41:12,956:DEBUG::
2022-04-27 06:41:26,304:ERROR::UpdatePasswordError: Failed to Update Password,It is required that you pass in a value for the "algorithms" argument when calling decode().
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1153, in Update_Password
    data = jwt.decode(token,os.environ.get('SECRETE_KEY'))
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 119, in decode
    decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 86, in decode_complete
    raise DecodeError(
jwt.exceptions.DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-04-27 06:43:01,686:ERROR::ResetPasswordError: Failed to Reset Password,'str' object has no attribute 'decode'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1142, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    return {"ResetToken":token.decode("UTF-8"),"status_message":"Password Reset Mail Sent","status":"success","status_code":200}
AttributeError: 'str' object has no attribute 'decode'
2022-04-27 06:43:58,074:DEBUG::
2022-04-27 06:43:58,075:DEBUG::====================================================================================================
2022-04-27 06:43:58,075:INFO::Server Utils Section :: Logging Active
2022-04-27 06:43:58,075:DEBUG::
2022-04-27 06:44:55,587:ERROR::UpdatePasswordError: Failed to Update Password,It is required that you pass in a value for the "algorithms" argument when calling decode().
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1153, in Update_Password
    data = jwt.decode(token,os.environ.get('SECRETE_KEY'))
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 119, in decode
    decoded = self.decode_complete(jwt, key, algorithms, options, **kwargs)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\jwt\api_jwt.py", line 86, in decode_complete
    raise DecodeError(
jwt.exceptions.DecodeError: It is required that you pass in a value for the "algorithms" argument when calling decode().
2022-04-27 14:18:45,462:DEBUG::
2022-04-27 14:18:45,462:DEBUG::====================================================================================================
2022-04-27 14:18:45,463:INFO::Server Utils Section :: Logging Active
2022-04-27 14:18:45,463:DEBUG::
2022-04-27 14:19:18,351:ERROR::ResetPasswordError: Failed to Reset Password,'NoneType' object has no attribute 'admin_reset_pin'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1126, in Reset_Password
    admin.admin_reset_pin = reset_pin
AttributeError: 'NoneType' object has no attribute 'admin_reset_pin'
2022-04-27 14:20:00,348:ERROR::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000224408CB760>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x00000224408CB760>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1142, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1091, in Send_Reset_Mail
    api_response = sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 162, in request
    r = self.pool_manager.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 78, in request
    return self.request_encode_body(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000224408CB760>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 14:22:34,244:ERROR::UpdatePasswordError: Failed to Update Password,'>' not supported between instances of 'datetime.datetime' and 'str'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1158, in Update_Password
    if datetime.datetime.now() > expiry_time:
TypeError: '>' not supported between instances of 'datetime.datetime' and 'str'
2022-04-27 14:23:59,807:DEBUG::
2022-04-27 14:23:59,807:DEBUG::====================================================================================================
2022-04-27 14:23:59,807:INFO::Server Utils Section :: Logging Active
2022-04-27 14:23:59,808:DEBUG::
2022-04-27 14:24:03,247:ERROR::UpdatePasswordError: Failed to Update Password,'>' not supported between instances of 'datetime.datetime' and 'str'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1159, in Update_Password
    if datetime.datetime.now() > expiry_time:
TypeError: '>' not supported between instances of 'datetime.datetime' and 'str'
2022-04-27 14:25:50,462:DEBUG::
2022-04-27 14:25:50,462:DEBUG::====================================================================================================
2022-04-27 14:25:50,463:INFO::Server Utils Section :: Logging Active
2022-04-27 14:25:50,463:DEBUG::
2022-04-27 14:25:57,945:ERROR::UpdatePasswordError: Failed to Update Password,strptime() takes exactly 2 arguments (1 given)
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1155, in Update_Password
    expiry_time = datetime.datetime.strptime(data['expiration'])
TypeError: strptime() takes exactly 2 arguments (1 given)
2022-04-27 14:27:11,540:DEBUG::
2022-04-27 14:27:11,541:DEBUG::====================================================================================================
2022-04-27 14:27:11,541:INFO::Server Utils Section :: Logging Active
2022-04-27 14:27:11,542:DEBUG::
2022-04-27 14:31:52,194:DEBUG::
2022-04-27 14:31:52,194:DEBUG::====================================================================================================
2022-04-27 14:31:52,194:INFO::Server Utils Section :: Logging Active
2022-04-27 14:31:52,195:DEBUG::
2022-04-27 14:32:58,809:DEBUG::
2022-04-27 14:32:58,809:DEBUG::====================================================================================================
2022-04-27 14:32:58,810:INFO::Server Utils Section :: Logging Active
2022-04-27 14:32:58,810:DEBUG::
2022-04-27 14:38:19,495:DEBUG::
2022-04-27 14:38:19,496:DEBUG::====================================================================================================
2022-04-27 14:38:19,496:INFO::Server Utils Section :: Logging Active
2022-04-27 14:38:19,496:DEBUG::
2022-04-27 14:48:49,724:DEBUG::
2022-04-27 14:48:49,724:DEBUG::====================================================================================================
2022-04-27 14:48:49,724:INFO::Server Utils Section :: Logging Active
2022-04-27 14:48:49,725:DEBUG::
2022-04-27 14:52:23,790:DEBUG::
2022-04-27 14:52:23,791:DEBUG::====================================================================================================
2022-04-27 14:52:23,791:INFO::Server Utils Section :: Logging Active
2022-04-27 14:52:23,792:DEBUG::
2022-04-27 15:39:21,569:DEBUG::
2022-04-27 15:39:21,570:DEBUG::====================================================================================================
2022-04-27 15:39:21,570:INFO::Server Utils Section :: Logging Active
2022-04-27 15:39:21,570:DEBUG::
2022-04-27 16:19:37,545:DEBUG::
2022-04-27 16:19:37,545:DEBUG::====================================================================================================
2022-04-27 16:19:37,546:INFO::Server Utils Section :: Logging Active
2022-04-27 16:19:37,546:DEBUG::
2022-04-27 16:20:27,839:DEBUG::
2022-04-27 16:20:27,839:DEBUG::====================================================================================================
2022-04-27 16:20:27,839:INFO::Server Utils Section :: Logging Active
2022-04-27 16:20:27,840:DEBUG::
2022-04-27 16:20:36,080:ERROR::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC7A30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x0000022E58DC7A30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1149, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 162, in request
    r = self.pool_manager.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 78, in request
    return self.request_encode_body(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC7A30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 16:20:48,320:ERROR::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC6E30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x0000022E58DC6E30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1149, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 162, in request
    r = self.pool_manager.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 78, in request
    return self.request_encode_body(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC6E30>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 16:20:57,215:ERROR::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC6590>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x0000022E58DC6590>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1149, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 162, in request
    r = self.pool_manager.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 78, in request
    return self.request_encode_body(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000022E58DC6590>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 16:55:09,875:DEBUG::
2022-04-27 16:55:09,875:DEBUG::====================================================================================================
2022-04-27 16:55:09,875:INFO::Server Utils Section :: Logging Active
2022-04-27 16:55:09,877:DEBUG::
2022-04-27 18:14:51,445:DEBUG::
2022-04-27 18:14:51,446:DEBUG::====================================================================================================
2022-04-27 18:14:51,446:INFO::Server Utils Section :: Logging Active
2022-04-27 18:14:51,446:DEBUG::
2022-04-27 18:16:45,101:ERROR::ResetPasswordError: Failed to Reset Password,HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001E537117A60>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 174, in _new_conn
    conn = connection.create_connection(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\connection.py", line 72, in create_connection
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 386, in _make_request
    self._validate_conn(conn)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 1040, in _validate_conn
    conn.connect()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 358, in connect
    self.sock = conn = self._new_conn()
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connection.py", line 186, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0x000001E537117A60>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1149, in Reset_Password
    Send_Response = Send_Reset_Mail(email,reset_pin)
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 162, in request
    r = self.pool_manager.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 78, in request
    return self.request_encode_body(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\request.py", line 170, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\poolmanager.py", line 376, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 813, in urlopen
    return self.urlopen(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\connectionpool.py", line 785, in urlopen
    retries = retries.increment(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\urllib3\util\retry.py", line 592, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.sendinblue.com', port=443): Max retries exceeded with url: /v3/smtp/email (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001E537117A60>: Failed to establish a new connection: [Errno 11002] getaddrinfo failed'))
2022-04-27 18:16:51,154:ERROR::SendResetMailError: Failed to Send Reset Mail,(403)
Reason: Forbidden
HTTP response headers: HTTPHeaderDict({'Date': 'Wed, 27 Apr 2022 16:16:51 GMT', 'Content-Type': 'application/json', 'Content-Length': '169', 'Connection': 'keep-alive', 'sib-request-id': 'dcc43aee-fd45-9ec1-af20-51ff8ee67195', 'access-control-allow-origin': '*', 'access-control-allow-credentials': 'true', 'access-control-allow-headers': '*', 'access-control-allow-methods': 'PUT, POST, GET, DELETE, PATCH, OPTIONS', 'x-envoy-upstream-service-time': '6', 'x-sib-server': 'srv-pr-online-rancher-worker-32.onl.51b.tech', 'x-content-type-options': 'nosniff', 'x-xss-protection': '1', 'CF-Cache-Status': 'DYNAMIC', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '7028dd6b2f3a5cff-DUR'})
HTTP response body: {"message":"Unable to send email. Your SMTP account is not yet activated. Please contact us at contact@sendinblue.com to request activation","code":"permission_denied"}

Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1098, in Send_Reset_Mail
    sendinblue_api.send_transac_email(send_smtp_email)
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1642, in send_transac_email
    (data) = self.send_transac_email_with_http_info(send_smtp_email, **kwargs)  # noqa: E501
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api\transactional_emails_api.py", line 1705, in send_transac_email_with_http_info
    return self.api_client.call_api(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 329, in call_api
    return self.__call_api(resource_path, method,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 161, in __call_api
    response_data = self.request(
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\api_client.py", line 371, in request
    return self.rest_client.POST(url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 269, in POST
    return self.request("POST", url,
  File "C:\Users\Adefolahan\miniconda3\envs\doxael\lib\site-packages\sib_api_v3_sdk\rest.py", line 228, in request
    raise ApiException(http_resp=r)
sib_api_v3_sdk.rest.ApiException: (403)
Reason: Forbidden
HTTP response headers: HTTPHeaderDict({'Date': 'Wed, 27 Apr 2022 16:16:51 GMT', 'Content-Type': 'application/json', 'Content-Length': '169', 'Connection': 'keep-alive', 'sib-request-id': 'dcc43aee-fd45-9ec1-af20-51ff8ee67195', 'access-control-allow-origin': '*', 'access-control-allow-credentials': 'true', 'access-control-allow-headers': '*', 'access-control-allow-methods': 'PUT, POST, GET, DELETE, PATCH, OPTIONS', 'x-envoy-upstream-service-time': '6', 'x-sib-server': 'srv-pr-online-rancher-worker-32.onl.51b.tech', 'x-content-type-options': 'nosniff', 'x-xss-protection': '1', 'CF-Cache-Status': 'DYNAMIC', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '7028dd6b2f3a5cff-DUR'})
HTTP response body: {"message":"Unable to send email. Your SMTP account is not yet activated. Please contact us at contact@sendinblue.com to request activation","code":"permission_denied"}


2022-05-02 00:26:06,765:DEBUG::
2022-05-02 00:26:06,766:DEBUG::====================================================================================================
2022-05-02 00:26:06,766:INFO::Server Utils Section :: Logging Active
2022-05-02 00:26:06,766:DEBUG::
2022-05-02 00:26:57,727:DEBUG::
2022-05-02 00:26:57,728:DEBUG::====================================================================================================
2022-05-02 00:26:57,728:INFO::Server Utils Section :: Logging Active
2022-05-02 00:26:57,728:DEBUG::
2022-05-02 00:28:47,748:DEBUG::
2022-05-02 00:28:47,748:DEBUG::====================================================================================================
2022-05-02 00:28:47,749:INFO::Server Utils Section :: Logging Active
2022-05-02 00:28:47,749:DEBUG::
2022-05-02 00:32:00,479:DEBUG::
2022-05-02 00:32:00,480:DEBUG::====================================================================================================
2022-05-02 00:32:00,480:INFO::Server Utils Section :: Logging Active
2022-05-02 00:32:00,481:DEBUG::
2022-05-02 00:32:33,761:DEBUG::
2022-05-02 00:32:33,762:DEBUG::====================================================================================================
2022-05-02 00:32:33,762:INFO::Server Utils Section :: Logging Active
2022-05-02 00:32:33,762:DEBUG::
2022-05-02 00:33:06,370:DEBUG::
2022-05-02 00:33:06,371:DEBUG::====================================================================================================
2022-05-02 00:33:06,371:INFO::Server Utils Section :: Logging Active
2022-05-02 00:33:06,371:DEBUG::
2022-05-02 00:35:05,229:DEBUG::
2022-05-02 00:35:05,229:DEBUG::====================================================================================================
2022-05-02 00:35:05,229:INFO::Server Utils Section :: Logging Active
2022-05-02 00:35:05,230:DEBUG::
2022-05-02 00:41:37,692:DEBUG::
2022-05-02 00:41:37,692:DEBUG::====================================================================================================
2022-05-02 00:41:37,693:INFO::Server Utils Section :: Logging Active
2022-05-02 00:41:37,693:DEBUG::
2022-05-02 00:41:47,157:DEBUG::
2022-05-02 00:41:47,157:DEBUG::====================================================================================================
2022-05-02 00:41:47,158:INFO::Server Utils Section :: Logging Active
2022-05-02 00:41:47,158:DEBUG::
2022-05-02 00:43:02,516:DEBUG::
2022-05-02 00:43:02,517:DEBUG::====================================================================================================
2022-05-02 00:43:02,518:INFO::Server Utils Section :: Logging Active
2022-05-02 00:43:02,519:DEBUG::
2022-05-02 00:43:07,078:DEBUG::
2022-05-02 00:43:07,078:DEBUG::====================================================================================================
2022-05-02 00:43:07,079:INFO::Server Utils Section :: Logging Active
2022-05-02 00:43:07,079:DEBUG::
2022-05-02 00:43:15,268:DEBUG::
2022-05-02 00:43:15,269:DEBUG::====================================================================================================
2022-05-02 00:43:15,269:INFO::Server Utils Section :: Logging Active
2022-05-02 00:43:15,269:DEBUG::
2022-05-02 00:45:46,713:DEBUG::
2022-05-02 00:45:46,713:DEBUG::====================================================================================================
2022-05-02 00:45:46,714:INFO::Server Utils Section :: Logging Active
2022-05-02 00:45:46,714:DEBUG::
2022-05-02 00:48:20,776:DEBUG::
2022-05-02 00:48:20,776:DEBUG::====================================================================================================
2022-05-02 00:48:20,776:INFO::Server Utils Section :: Logging Active
2022-05-02 00:48:20,777:DEBUG::
2022-05-02 00:49:40,461:DEBUG::
2022-05-02 00:49:40,461:DEBUG::====================================================================================================
2022-05-02 00:49:40,461:INFO::Server Utils Section :: Logging Active
2022-05-02 00:49:40,462:DEBUG::
2022-05-02 00:50:48,638:DEBUG::
2022-05-02 00:50:48,638:DEBUG::====================================================================================================
2022-05-02 00:50:48,638:INFO::Server Utils Section :: Logging Active
2022-05-02 00:50:48,639:DEBUG::
2022-05-02 00:53:54,913:DEBUG::
2022-05-02 00:53:54,914:DEBUG::====================================================================================================
2022-05-02 00:53:54,914:INFO::Server Utils Section :: Logging Active
2022-05-02 00:53:54,915:DEBUG::
2022-05-02 00:55:45,411:DEBUG::
2022-05-02 00:55:45,412:DEBUG::====================================================================================================
2022-05-02 00:55:45,412:INFO::Server Utils Section :: Logging Active
2022-05-02 00:55:45,412:DEBUG::
2022-05-02 00:57:11,063:DEBUG::
2022-05-02 00:57:11,063:DEBUG::====================================================================================================
2022-05-02 00:57:11,064:INFO::Server Utils Section :: Logging Active
2022-05-02 00:57:11,064:DEBUG::
2022-05-02 00:57:37,438:DEBUG::
2022-05-02 00:57:37,438:DEBUG::====================================================================================================
2022-05-02 00:57:37,439:INFO::Server Utils Section :: Logging Active
2022-05-02 00:57:37,439:DEBUG::
2022-05-02 01:16:17,407:DEBUG::
2022-05-02 01:16:17,407:DEBUG::====================================================================================================
2022-05-02 01:16:17,408:INFO::Server Utils Section :: Logging Active
2022-05-02 01:16:17,408:DEBUG::
2022-05-02 01:27:57,832:DEBUG::
2022-05-02 01:27:57,832:DEBUG::====================================================================================================
2022-05-02 01:27:57,832:INFO::Server Utils Section :: Logging Active
2022-05-02 01:27:57,833:DEBUG::
2022-05-02 01:28:07,503:ERROR::ShowPurchasesError: Failed to Show Purchases,can't multiply sequence by non-int of type 'str'
Traceback (most recent call last):
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1308, in Show_Purchases
    orders_list = [{"order_part_of":order.order_id,"order_vendor_id":order.order_vendor_id,"item_unique_id":order.item_unique_id,"order_total_price":order.order_total_price,"order_product_id":order.order_product_id,"order_product_name":order.order_product_name,"order_product_price":order.order_product_price,"order_product_discount":order.order_product_discount,"order_product_quantity":order.order_product_quantity,"order_product_image":order.order_product_image,"order_product_description":order.order_product_description,"order_product_was_available":order.order_product_is_available,"order_date":order.order_date,"item_order_total_value":((order.order_product_price * order.order_product_quantity) - ((order.order_product_discount/100) * (order.order_product_price * order.order_product_quantity)))} for order in orders]
  File "C:\Users\Adefolahan\WEB_APPS\Doxael\Api_Server\app\server_utils.py", line 1308, in <listcomp>
    orders_list = [{"order_part_of":order.order_id,"order_vendor_id":order.order_vendor_id,"item_unique_id":order.item_unique_id,"order_total_price":order.order_total_price,"order_product_id":order.order_product_id,"order_product_name":order.order_product_name,"order_product_price":order.order_product_price,"order_product_discount":order.order_product_discount,"order_product_quantity":order.order_product_quantity,"order_product_image":order.order_product_image,"order_product_description":order.order_product_description,"order_product_was_available":order.order_product_is_available,"order_date":order.order_date,"item_order_total_value":((order.order_product_price * order.order_product_quantity) - ((order.order_product_discount/100) * (order.order_product_price * order.order_product_quantity)))} for order in orders]
TypeError: can't multiply sequence by non-int of type 'str'
2022-05-02 01:29:34,089:DEBUG::
2022-05-02 01:29:34,090:DEBUG::====================================================================================================
2022-05-02 01:29:34,090:INFO::Server Utils Section :: Logging Active
2022-05-02 01:29:34,090:DEBUG::
2022-05-02 03:10:40,979:DEBUG::
2022-05-02 03:10:40,980:DEBUG::====================================================================================================
2022-05-02 03:10:40,980:INFO::Server Utils Section :: Logging Active
2022-05-02 03:10:40,981:DEBUG::
2022-07-06 10:19:59,079:DEBUG::
2022-07-06 10:19:59,088:DEBUG::====================================================================================================
2022-07-06 10:19:59,089:INFO::Server Utils Section :: Logging Active
2022-07-06 10:19:59,091:DEBUG::
2022-07-06 10:23:27,343:DEBUG::
2022-07-06 10:23:27,344:DEBUG::====================================================================================================
2022-07-06 10:23:27,344:INFO::Server Utils Section :: Logging Active
2022-07-06 10:23:27,344:DEBUG::
2022-07-06 10:24:58,484:DEBUG::
2022-07-06 10:24:58,484:DEBUG::====================================================================================================
2022-07-06 10:24:58,484:INFO::Server Utils Section :: Logging Active
2022-07-06 10:24:58,484:DEBUG::
2022-07-06 10:29:25,126:DEBUG::
2022-07-06 10:29:25,127:DEBUG::====================================================================================================
2022-07-06 10:29:25,127:INFO::Server Utils Section :: Logging Active
2022-07-06 10:29:25,127:DEBUG::
2022-07-06 11:20:41,556:DEBUG::
2022-07-06 11:20:41,557:DEBUG::====================================================================================================
2022-07-06 11:20:41,557:INFO::Server Utils Section :: Logging Active
2022-07-06 11:20:41,557:DEBUG::
2022-07-06 11:25:28,977:DEBUG::
2022-07-06 11:25:28,978:DEBUG::====================================================================================================
2022-07-06 11:25:29,002:INFO::Server Utils Section :: Logging Active
2022-07-06 11:25:29,002:DEBUG::
2022-07-06 11:26:58,254:DEBUG::
2022-07-06 11:26:58,254:DEBUG::====================================================================================================
2022-07-06 11:26:58,254:INFO::Server Utils Section :: Logging Active
2022-07-06 11:26:58,254:DEBUG::
2022-07-06 11:27:56,734:DEBUG::
2022-07-06 11:27:56,734:DEBUG::====================================================================================================
2022-07-06 11:27:56,736:INFO::Server Utils Section :: Logging Active
2022-07-06 11:27:56,736:DEBUG::
2022-07-06 11:28:06,937:DEBUG::
2022-07-06 11:28:06,938:DEBUG::====================================================================================================
2022-07-06 11:28:06,940:INFO::Server Utils Section :: Logging Active
2022-07-06 11:28:06,941:DEBUG::
2022-07-06 11:28:16,988:DEBUG::
2022-07-06 11:28:16,988:DEBUG::====================================================================================================
2022-07-06 11:28:16,989:INFO::Server Utils Section :: Logging Active
2022-07-06 11:28:16,989:DEBUG::
2022-07-06 11:28:25,624:DEBUG::
2022-07-06 11:28:25,624:DEBUG::====================================================================================================
2022-07-06 11:28:25,625:INFO::Server Utils Section :: Logging Active
2022-07-06 11:28:25,626:DEBUG::
2022-07-07 10:08:21,790:DEBUG::
2022-07-07 10:08:21,805:DEBUG::====================================================================================================
2022-07-07 10:08:21,805:INFO::Server Utils Section :: Logging Active
2022-07-07 10:08:21,805:DEBUG::
2022-07-08 10:09:28,655:DEBUG::
2022-07-08 10:09:28,669:DEBUG::====================================================================================================
2022-07-08 10:09:28,669:INFO::Server Utils Section :: Logging Active
2022-07-08 10:09:28,670:DEBUG::
2022-07-11 11:49:30,716:DEBUG::
2022-07-11 11:49:30,730:DEBUG::====================================================================================================
2022-07-11 11:49:30,730:INFO::Server Utils Section :: Logging Active
2022-07-11 11:49:30,731:DEBUG::
2022-07-12 10:39:48,152:DEBUG::
2022-07-12 10:39:48,163:DEBUG::====================================================================================================
2022-07-12 10:39:48,164:INFO::Server Utils Section :: Logging Active
2022-07-12 10:39:48,164:DEBUG::
2022-07-13 14:37:42,724:DEBUG::
2022-07-13 14:37:42,732:DEBUG::====================================================================================================
2022-07-13 14:37:42,732:INFO::Server Utils Section :: Logging Active
2022-07-13 14:37:42,732:DEBUG::
2022-07-14 09:31:41,641:DEBUG::
2022-07-14 09:31:41,677:DEBUG::====================================================================================================
2022-07-14 09:31:41,678:INFO::Server Utils Section :: Logging Active
2022-07-14 09:31:41,678:DEBUG::
2022-07-14 18:36:53,533:DEBUG::
2022-07-14 18:36:53,546:DEBUG::====================================================================================================
2022-07-14 18:36:53,546:INFO::Server Utils Section :: Logging Active
2022-07-14 18:36:53,546:DEBUG::
2022-07-18 10:18:06,421:DEBUG::
2022-07-18 10:18:06,435:DEBUG::====================================================================================================
2022-07-18 10:18:06,435:INFO::Server Utils Section :: Logging Active
2022-07-18 10:18:06,435:DEBUG::
2022-07-19 18:21:54,031:DEBUG::
2022-07-19 18:21:54,049:DEBUG::====================================================================================================
2022-07-19 18:21:54,049:INFO::Server Utils Section :: Logging Active
2022-07-19 18:21:54,050:DEBUG::
2022-07-21 11:27:08,331:DEBUG::
2022-07-21 11:27:08,337:DEBUG::====================================================================================================
2022-07-21 11:27:08,337:INFO::Server Utils Section :: Logging Active
2022-07-21 11:27:08,337:DEBUG::
2022-07-21 11:27:26,711:DEBUG::
2022-07-21 11:27:26,711:DEBUG::====================================================================================================
2022-07-21 11:27:26,711:INFO::Server Utils Section :: Logging Active
2022-07-21 11:27:26,712:DEBUG::
2022-07-21 11:31:37,479:DEBUG::
2022-07-21 11:31:37,480:DEBUG::====================================================================================================
2022-07-21 11:31:37,480:INFO::Server Utils Section :: Logging Active
2022-07-21 11:31:37,480:DEBUG::
2022-07-21 11:42:25,464:DEBUG::
2022-07-21 11:42:25,464:DEBUG::====================================================================================================
2022-07-21 11:42:25,464:INFO::Server Utils Section :: Logging Active
2022-07-21 11:42:25,465:DEBUG::
2022-07-21 11:57:15,516:DEBUG::
2022-07-21 11:57:15,516:DEBUG::====================================================================================================
2022-07-21 11:57:15,517:INFO::Server Utils Section :: Logging Active
2022-07-21 11:57:15,517:DEBUG::
2022-07-21 12:18:37,112:DEBUG::EditProductError: Failed to Edit Product,[WinError 3] The system cannot find the path specified: 'D:\\React\\PL\\#10182_vishnu_Grand sprorts\\backend-main\\backend-main\\app\\static/images/products/a1affe613d4df7fd3bee3558c3e532f7_Screenshot_1.png'
2022-07-21 12:24:57,814:DEBUG::
2022-07-21 12:24:57,814:DEBUG::====================================================================================================
2022-07-21 12:24:57,814:INFO::Server Utils Section :: Logging Active
2022-07-21 12:24:57,815:DEBUG::
2022-07-21 12:25:55,954:DEBUG::
2022-07-21 12:25:55,955:DEBUG::====================================================================================================
2022-07-21 12:25:55,955:INFO::Server Utils Section :: Logging Active
2022-07-21 12:25:55,955:DEBUG::
2022-07-21 12:30:39,759:DEBUG::EditProductError: Failed to Edit Product,[WinError 3] The system cannot find the path specified: 'D:\\React\\PL\\#10182_vishnu_Grand sprorts\\backend-main\\backend-main\\app\\static/images/products/a1affe613d4df7fd3bee3558c3e532f7_Screenshot_1.png'
2022-07-21 12:36:13,854:DEBUG::
2022-07-21 12:36:13,855:DEBUG::====================================================================================================
2022-07-21 12:36:13,855:INFO::Server Utils Section :: Logging Active
2022-07-21 12:36:13,856:DEBUG::
2022-07-21 12:59:25,215:DEBUG::
2022-07-21 12:59:25,215:DEBUG::====================================================================================================
2022-07-21 12:59:25,215:INFO::Server Utils Section :: Logging Active
2022-07-21 12:59:25,216:DEBUG::
2022-07-22 11:34:11,881:DEBUG::
2022-07-22 11:34:11,896:DEBUG::====================================================================================================
2022-07-22 11:34:11,896:INFO::Server Utils Section :: Logging Active
2022-07-22 11:34:11,896:DEBUG::
2022-07-22 11:34:24,100:DEBUG::
2022-07-22 11:34:24,100:DEBUG::====================================================================================================
2022-07-22 11:34:24,101:INFO::Server Utils Section :: Logging Active
2022-07-22 11:34:24,101:DEBUG::
2022-07-22 11:34:37,718:DEBUG::
2022-07-22 11:34:37,718:DEBUG::====================================================================================================
2022-07-22 11:34:37,719:INFO::Server Utils Section :: Logging Active
2022-07-22 11:34:37,721:DEBUG::
2022-07-22 11:34:39,092:DEBUG::
2022-07-22 11:34:39,093:DEBUG::====================================================================================================
2022-07-22 11:34:39,094:INFO::Server Utils Section :: Logging Active
2022-07-22 11:34:39,096:DEBUG::
2022-07-22 11:45:16,974:DEBUG::
2022-07-22 11:45:16,974:DEBUG::====================================================================================================
2022-07-22 11:45:16,974:INFO::Server Utils Section :: Logging Active
2022-07-22 11:45:16,974:DEBUG::
2022-07-22 12:18:37,715:DEBUG::
2022-07-22 12:18:37,715:DEBUG::====================================================================================================
2022-07-22 12:18:37,716:INFO::Server Utils Section :: Logging Active
2022-07-22 12:18:37,716:DEBUG::
2022-07-22 15:25:29,708:DEBUG::
2022-07-22 15:25:29,708:DEBUG::====================================================================================================
2022-07-22 15:25:29,710:INFO::Server Utils Section :: Logging Active
2022-07-22 15:25:29,712:DEBUG::
2022-07-25 14:09:06,824:DEBUG::
2022-07-25 14:09:06,825:DEBUG::====================================================================================================
2022-07-25 14:09:06,825:INFO::Server Utils Section :: Logging Active
2022-07-25 14:09:06,825:DEBUG::
2022-07-25 15:53:20,636:DEBUG::
2022-07-25 15:53:20,636:DEBUG::====================================================================================================
2022-07-25 15:53:20,636:INFO::Server Utils Section :: Logging Active
2022-07-25 15:53:20,636:DEBUG::
2022-07-25 16:21:07,894:DEBUG::
2022-07-25 16:21:07,895:DEBUG::====================================================================================================
2022-07-25 16:21:07,896:INFO::Server Utils Section :: Logging Active
2022-07-25 16:21:07,897:DEBUG::
2022-07-25 16:22:20,049:DEBUG::
2022-07-25 16:22:20,050:DEBUG::====================================================================================================
2022-07-25 16:22:20,050:INFO::Server Utils Section :: Logging Active
2022-07-25 16:22:20,050:DEBUG::
2022-07-25 16:22:54,744:DEBUG::
2022-07-25 16:22:54,744:DEBUG::====================================================================================================
2022-07-25 16:22:54,745:INFO::Server Utils Section :: Logging Active
2022-07-25 16:22:54,747:DEBUG::
2022-07-25 16:23:04,564:DEBUG::
2022-07-25 16:23:04,564:DEBUG::====================================================================================================
2022-07-25 16:23:04,565:INFO::Server Utils Section :: Logging Active
2022-07-25 16:23:04,567:DEBUG::
2022-07-25 16:23:14,444:DEBUG::
2022-07-25 16:23:14,445:DEBUG::====================================================================================================
2022-07-25 16:23:14,449:INFO::Server Utils Section :: Logging Active
2022-07-25 16:23:14,450:DEBUG::
2022-07-25 16:23:31,561:DEBUG::
2022-07-25 16:23:31,561:DEBUG::====================================================================================================
2022-07-25 16:23:31,561:INFO::Server Utils Section :: Logging Active
2022-07-25 16:23:31,561:DEBUG::
2022-07-25 16:23:51,796:DEBUG::
2022-07-25 16:23:51,797:DEBUG::====================================================================================================
2022-07-25 16:23:51,798:INFO::Server Utils Section :: Logging Active
2022-07-25 16:23:51,800:DEBUG::
2022-07-25 16:23:56,703:DEBUG::
2022-07-25 16:23:56,703:DEBUG::====================================================================================================
2022-07-25 16:23:56,704:INFO::Server Utils Section :: Logging Active
2022-07-25 16:23:56,704:DEBUG::
2022-07-25 16:35:03,881:DEBUG::AddNewProductError: Failed to Add New Product,(MySQLdb.OperationalError) (2002, "Can't connect to server on 'sql6.freemysqlhosting.net' (10060)")
(Background on this error at: https://sqlalche.me/e/14/e3q8)
2022-07-25 16:36:28,983:DEBUG::RemoveProductError: Failed to Remove Product,local variable 'product' referenced before assignment
2022-07-25 16:36:33,558:DEBUG::RemoveProductError: Failed to Remove Product,local variable 'product' referenced before assignment
2022-07-25 16:36:35,264:DEBUG::RemoveProductError: Failed to Remove Product,local variable 'product' referenced before assignment
2022-07-25 16:36:58,430:DEBUG::RemoveProductError: Failed to Remove Product,local variable 'product' referenced before assignment
2022-07-27 11:47:17,090:DEBUG::
2022-07-27 11:47:17,090:DEBUG::====================================================================================================
2022-07-27 11:47:17,090:INFO::Server Utils Section :: Logging Active
2022-07-27 11:47:17,090:DEBUG::
2022-07-27 11:49:05,505:DEBUG::
2022-07-27 11:49:05,505:DEBUG::====================================================================================================
2022-07-27 11:49:05,505:INFO::Server Utils Section :: Logging Active
2022-07-27 11:49:05,505:DEBUG::
2022-07-27 11:52:13,619:DEBUG::
2022-07-27 11:52:13,620:DEBUG::====================================================================================================
2022-07-27 11:52:13,621:INFO::Server Utils Section :: Logging Active
2022-07-27 11:52:13,623:DEBUG::
2022-07-27 11:55:01,375:DEBUG::
2022-07-27 11:55:01,375:DEBUG::====================================================================================================
2022-07-27 11:55:01,376:INFO::Server Utils Section :: Logging Active
2022-07-27 11:55:01,378:DEBUG::
2022-07-27 12:04:45,429:DEBUG::
2022-07-27 12:04:45,430:DEBUG::====================================================================================================
2022-07-27 12:04:45,431:INFO::Server Utils Section :: Logging Active
2022-07-27 12:04:45,432:DEBUG::
2022-07-27 12:05:41,248:DEBUG::
2022-07-27 12:05:41,249:DEBUG::====================================================================================================
2022-07-27 12:05:41,249:INFO::Server Utils Section :: Logging Active
2022-07-27 12:05:41,250:DEBUG::
2022-07-27 12:06:29,453:DEBUG::
2022-07-27 12:06:29,453:DEBUG::====================================================================================================
2022-07-27 12:06:29,456:INFO::Server Utils Section :: Logging Active
2022-07-27 12:06:29,457:DEBUG::
2022-07-27 12:06:48,918:DEBUG::
2022-07-27 12:06:48,918:DEBUG::====================================================================================================
2022-07-27 12:06:48,919:INFO::Server Utils Section :: Logging Active
2022-07-27 12:06:48,921:DEBUG::
2022-07-27 12:09:47,730:DEBUG::
2022-07-27 12:09:47,730:DEBUG::====================================================================================================
2022-07-27 12:09:47,732:INFO::Server Utils Section :: Logging Active
2022-07-27 12:09:47,733:DEBUG::
2022-07-27 12:16:58,616:DEBUG::
2022-07-27 12:16:58,616:DEBUG::====================================================================================================
2022-07-27 12:16:58,616:INFO::Server Utils Section :: Logging Active
2022-07-27 12:16:58,617:DEBUG::
2022-07-27 12:18:33,749:DEBUG::
2022-07-27 12:18:33,749:DEBUG::====================================================================================================
2022-07-27 12:18:33,749:INFO::Server Utils Section :: Logging Active
2022-07-27 12:18:33,749:DEBUG::
2022-07-27 12:20:33,757:DEBUG::
2022-07-27 12:20:33,757:DEBUG::====================================================================================================
2022-07-27 12:20:33,758:INFO::Server Utils Section :: Logging Active
2022-07-27 12:20:33,760:DEBUG::
2022-07-27 12:20:59,555:DEBUG::
2022-07-27 12:20:59,555:DEBUG::====================================================================================================
2022-07-27 12:20:59,556:INFO::Server Utils Section :: Logging Active
2022-07-27 12:20:59,558:DEBUG::
2022-07-27 12:22:10,697:DEBUG::
2022-07-27 12:22:10,697:DEBUG::====================================================================================================
2022-07-27 12:22:10,698:INFO::Server Utils Section :: Logging Active
2022-07-27 12:22:10,699:DEBUG::
2022-07-27 12:23:26,468:DEBUG::
2022-07-27 12:23:26,468:DEBUG::====================================================================================================
2022-07-27 12:23:26,470:INFO::Server Utils Section :: Logging Active
2022-07-27 12:23:26,471:DEBUG::
2022-07-27 14:12:18,546:DEBUG::
2022-07-27 14:12:18,546:DEBUG::====================================================================================================
2022-07-27 14:12:18,558:INFO::Server Utils Section :: Logging Active
2022-07-27 14:12:18,559:DEBUG::
2022-07-27 14:12:44,877:DEBUG::
2022-07-27 14:12:44,877:DEBUG::====================================================================================================
2022-07-27 14:12:44,878:INFO::Server Utils Section :: Logging Active
2022-07-27 14:12:44,880:DEBUG::
2022-07-27 14:16:55,384:DEBUG::
2022-07-27 14:16:55,384:DEBUG::====================================================================================================
2022-07-27 14:16:55,385:INFO::Server Utils Section :: Logging Active
2022-07-27 14:16:55,387:DEBUG::
2022-07-27 14:17:06,550:DEBUG::
2022-07-27 14:17:06,551:DEBUG::====================================================================================================
2022-07-27 14:17:06,551:INFO::Server Utils Section :: Logging Active
2022-07-27 14:17:06,553:DEBUG::
2022-07-27 14:17:27,289:DEBUG::
2022-07-27 14:17:27,289:DEBUG::====================================================================================================
2022-07-27 14:17:27,292:INFO::Server Utils Section :: Logging Active
2022-07-27 14:17:27,292:DEBUG::
2022-07-27 14:19:50,343:DEBUG::
2022-07-27 14:19:50,343:DEBUG::====================================================================================================
2022-07-27 14:19:50,344:INFO::Server Utils Section :: Logging Active
2022-07-27 14:19:50,346:DEBUG::
2022-07-27 14:19:59,963:DEBUG::
2022-07-27 14:19:59,963:DEBUG::====================================================================================================
2022-07-27 14:19:59,964:INFO::Server Utils Section :: Logging Active
2022-07-27 14:19:59,966:DEBUG::
2022-07-27 14:23:20,599:DEBUG::
2022-07-27 14:23:20,599:DEBUG::====================================================================================================
2022-07-27 14:23:20,601:INFO::Server Utils Section :: Logging Active
2022-07-27 14:23:20,602:DEBUG::
2022-07-27 14:26:41,824:DEBUG::
2022-07-27 14:26:41,825:DEBUG::====================================================================================================
2022-07-27 14:26:41,827:INFO::Server Utils Section :: Logging Active
2022-07-27 14:26:41,827:DEBUG::
2022-07-27 14:38:01,771:DEBUG::
2022-07-27 14:38:01,772:DEBUG::====================================================================================================
2022-07-27 14:38:01,772:INFO::Server Utils Section :: Logging Active
2022-07-27 14:38:01,775:DEBUG::
2022-07-27 14:38:21,133:DEBUG::
2022-07-27 14:38:21,134:DEBUG::====================================================================================================
2022-07-27 14:38:21,135:INFO::Server Utils Section :: Logging Active
2022-07-27 14:38:21,137:DEBUG::
2022-07-27 14:38:42,834:DEBUG::
2022-07-27 14:38:42,834:DEBUG::====================================================================================================
2022-07-27 14:38:42,834:INFO::Server Utils Section :: Logging Active
2022-07-27 14:38:42,834:DEBUG::
2022-07-27 14:42:55,195:DEBUG::
2022-07-27 14:42:55,195:DEBUG::====================================================================================================
2022-07-27 14:42:55,196:INFO::Server Utils Section :: Logging Active
2022-07-27 14:42:55,198:DEBUG::
2022-07-27 14:45:35,272:DEBUG::
2022-07-27 14:45:35,272:DEBUG::====================================================================================================
2022-07-27 14:45:35,276:INFO::Server Utils Section :: Logging Active
2022-07-27 14:45:35,283:DEBUG::
2022-07-27 14:46:05,865:DEBUG::
2022-07-27 14:46:05,866:DEBUG::====================================================================================================
2022-07-27 14:46:05,867:INFO::Server Utils Section :: Logging Active
2022-07-27 14:46:05,869:DEBUG::
2022-07-27 14:46:38,430:DEBUG::
2022-07-27 14:46:38,432:DEBUG::====================================================================================================
2022-07-27 14:46:38,437:INFO::Server Utils Section :: Logging Active
2022-07-27 14:46:38,438:DEBUG::
2022-07-27 14:47:34,835:DEBUG::
2022-07-27 14:47:34,836:DEBUG::====================================================================================================
2022-07-27 14:47:34,837:INFO::Server Utils Section :: Logging Active
2022-07-27 14:47:34,839:DEBUG::
2022-07-27 14:50:51,594:DEBUG::
2022-07-27 14:50:51,594:DEBUG::====================================================================================================
2022-07-27 14:50:51,595:INFO::Server Utils Section :: Logging Active
2022-07-27 14:50:51,597:DEBUG::
2022-07-27 14:51:04,308:DEBUG::
2022-07-27 14:51:04,309:DEBUG::====================================================================================================
2022-07-27 14:51:04,310:INFO::Server Utils Section :: Logging Active
2022-07-27 14:51:04,311:DEBUG::
2022-07-27 14:51:21,940:DEBUG::
2022-07-27 14:51:21,940:DEBUG::====================================================================================================
2022-07-27 14:51:21,943:INFO::Server Utils Section :: Logging Active
2022-07-27 14:51:21,945:DEBUG::
2022-07-27 14:53:12,358:DEBUG::
2022-07-27 14:53:12,359:DEBUG::====================================================================================================
2022-07-27 14:53:12,360:INFO::Server Utils Section :: Logging Active
2022-07-27 14:53:12,363:DEBUG::
2022-07-27 14:53:17,260:DEBUG::
2022-07-27 14:53:17,261:DEBUG::====================================================================================================
2022-07-27 14:53:17,262:INFO::Server Utils Section :: Logging Active
2022-07-27 14:53:17,264:DEBUG::
2022-07-27 14:54:20,851:DEBUG::
2022-07-27 14:54:20,884:DEBUG::====================================================================================================
2022-07-27 14:54:20,888:INFO::Server Utils Section :: Logging Active
2022-07-27 14:54:20,889:DEBUG::
2022-07-27 14:57:24,582:DEBUG::
2022-07-27 14:57:24,582:DEBUG::====================================================================================================
2022-07-27 14:57:24,583:INFO::Server Utils Section :: Logging Active
2022-07-27 14:57:24,585:DEBUG::
2022-07-27 14:57:52,647:DEBUG::
2022-07-27 14:57:52,647:DEBUG::====================================================================================================
2022-07-27 14:57:52,649:INFO::Server Utils Section :: Logging Active
2022-07-27 14:57:52,651:DEBUG::
2022-07-27 14:58:27,287:DEBUG::
2022-07-27 14:58:27,288:DEBUG::====================================================================================================
2022-07-27 14:58:27,289:INFO::Server Utils Section :: Logging Active
2022-07-27 14:58:27,290:DEBUG::
2022-07-27 14:59:05,772:DEBUG::
2022-07-27 14:59:05,772:DEBUG::====================================================================================================
2022-07-27 14:59:05,773:INFO::Server Utils Section :: Logging Active
2022-07-27 14:59:05,775:DEBUG::
2022-07-27 15:11:07,838:DEBUG::
2022-07-27 15:11:07,838:DEBUG::====================================================================================================
2022-07-27 15:11:07,838:INFO::Server Utils Section :: Logging Active
2022-07-27 15:11:07,839:DEBUG::
2022-07-27 15:12:05,653:DEBUG::
2022-07-27 15:12:05,654:DEBUG::====================================================================================================
2022-07-27 15:12:05,656:INFO::Server Utils Section :: Logging Active
2022-07-27 15:12:05,658:DEBUG::
2022-07-27 15:12:31,384:DEBUG::
2022-07-27 15:12:31,384:DEBUG::====================================================================================================
2022-07-27 15:12:31,387:INFO::Server Utils Section :: Logging Active
2022-07-27 15:12:31,387:DEBUG::
2022-07-27 15:12:59,711:DEBUG::
2022-07-27 15:12:59,712:DEBUG::====================================================================================================
2022-07-27 15:12:59,714:INFO::Server Utils Section :: Logging Active
2022-07-27 15:12:59,714:DEBUG::
2022-07-27 15:17:06,954:DEBUG::
2022-07-27 15:17:06,954:DEBUG::====================================================================================================
2022-07-27 15:17:06,957:INFO::Server Utils Section :: Logging Active
2022-07-27 15:17:06,958:DEBUG::
2022-07-27 15:22:59,557:DEBUG::
2022-07-27 15:22:59,558:DEBUG::====================================================================================================
2022-07-27 15:22:59,559:INFO::Server Utils Section :: Logging Active
2022-07-27 15:22:59,561:DEBUG::
2022-07-27 15:24:01,523:DEBUG::
2022-07-27 15:24:01,524:DEBUG::====================================================================================================
2022-07-27 15:24:01,524:INFO::Server Utils Section :: Logging Active
2022-07-27 15:24:01,524:DEBUG::
2022-07-27 15:25:05,224:DEBUG::
2022-07-27 15:25:05,225:DEBUG::====================================================================================================
2022-07-27 15:25:05,225:INFO::Server Utils Section :: Logging Active
2022-07-27 15:25:05,225:DEBUG::
2022-07-27 15:26:20,707:DEBUG::
2022-07-27 15:26:20,708:DEBUG::====================================================================================================
2022-07-27 15:26:20,709:INFO::Server Utils Section :: Logging Active
2022-07-27 15:26:20,711:DEBUG::
2022-07-27 15:27:51,347:DEBUG::
2022-07-27 15:27:51,347:DEBUG::====================================================================================================
2022-07-27 15:27:51,348:INFO::Server Utils Section :: Logging Active
2022-07-27 15:27:51,348:DEBUG::
2022-07-27 15:28:19,758:DEBUG::
2022-07-27 15:28:19,759:DEBUG::====================================================================================================
2022-07-27 15:28:19,760:INFO::Server Utils Section :: Logging Active
2022-07-27 15:28:19,760:DEBUG::
2022-07-27 15:28:51,525:DEBUG::
2022-07-27 15:28:51,525:DEBUG::====================================================================================================
2022-07-27 15:28:51,526:INFO::Server Utils Section :: Logging Active
2022-07-27 15:28:51,527:DEBUG::
2022-07-27 15:29:02,805:DEBUG::
2022-07-27 15:29:02,806:DEBUG::====================================================================================================
2022-07-27 15:29:02,807:INFO::Server Utils Section :: Logging Active
2022-07-27 15:29:02,809:DEBUG::
2022-07-27 15:31:19,069:DEBUG::
2022-07-27 15:31:19,069:DEBUG::====================================================================================================
2022-07-27 15:31:19,070:INFO::Server Utils Section :: Logging Active
2022-07-27 15:31:19,072:DEBUG::
2022-07-27 15:33:45,374:DEBUG::
2022-07-27 15:33:45,374:DEBUG::====================================================================================================
2022-07-27 15:33:45,377:INFO::Server Utils Section :: Logging Active
2022-07-27 15:33:45,377:DEBUG::
2022-07-27 15:35:58,162:DEBUG::
2022-07-27 15:35:58,169:DEBUG::====================================================================================================
2022-07-27 15:35:58,170:INFO::Server Utils Section :: Logging Active
2022-07-27 15:35:58,172:DEBUG::
2022-07-27 15:38:22,656:DEBUG::
2022-07-27 15:38:22,656:DEBUG::====================================================================================================
2022-07-27 15:38:22,657:INFO::Server Utils Section :: Logging Active
2022-07-27 15:38:22,659:DEBUG::
2022-07-27 15:38:31,440:DEBUG::
2022-07-27 15:38:31,450:DEBUG::====================================================================================================
2022-07-27 15:38:31,451:INFO::Server Utils Section :: Logging Active
2022-07-27 15:38:31,452:DEBUG::
2022-07-27 15:39:13,480:DEBUG::
2022-07-27 15:39:13,480:DEBUG::====================================================================================================
2022-07-27 15:39:13,482:INFO::Server Utils Section :: Logging Active
2022-07-27 15:39:13,483:DEBUG::
2022-07-27 15:49:16,270:DEBUG::
2022-07-27 15:49:16,270:DEBUG::====================================================================================================
2022-07-27 15:49:16,271:INFO::Server Utils Section :: Logging Active
2022-07-27 15:49:16,273:DEBUG::
2022-07-27 15:51:12,973:DEBUG::
2022-07-27 15:51:12,973:DEBUG::====================================================================================================
2022-07-27 15:51:12,974:INFO::Server Utils Section :: Logging Active
2022-07-27 15:51:12,976:DEBUG::
2022-07-27 16:36:44,158:DEBUG::
2022-07-27 16:36:44,159:DEBUG::====================================================================================================
2022-07-27 16:36:44,160:INFO::Server Utils Section :: Logging Active
2022-07-27 16:36:44,162:DEBUG::
2022-07-28 10:18:51,463:DEBUG::
2022-07-28 10:18:51,471:DEBUG::====================================================================================================
2022-07-28 10:18:51,472:INFO::Server Utils Section :: Logging Active
2022-07-28 10:18:51,472:DEBUG::
2022-07-29 09:22:14,470:DEBUG::
2022-07-29 09:22:14,480:DEBUG::====================================================================================================
2022-07-29 09:22:14,481:INFO::Server Utils Section :: Logging Active
2022-07-29 09:22:14,481:DEBUG::
2022-07-29 09:37:27,658:DEBUG::
2022-07-29 09:37:27,658:DEBUG::====================================================================================================
2022-07-29 09:37:27,659:INFO::Server Utils Section :: Logging Active
2022-07-29 09:37:27,662:DEBUG::
2022-08-01 09:22:03,011:DEBUG::
2022-08-01 09:22:03,033:DEBUG::====================================================================================================
2022-08-01 09:22:03,034:INFO::Server Utils Section :: Logging Active
2022-08-01 09:22:03,034:DEBUG::
2022-08-02 09:18:10,318:DEBUG::
2022-08-02 09:18:10,332:DEBUG::====================================================================================================
2022-08-02 09:18:10,332:INFO::Server Utils Section :: Logging Active
2022-08-02 09:18:10,332:DEBUG::
2022-08-02 09:22:28,557:DEBUG::
2022-08-02 09:22:28,557:DEBUG::====================================================================================================
2022-08-02 09:22:28,558:INFO::Server Utils Section :: Logging Active
2022-08-02 09:22:28,560:DEBUG::
2022-08-02 09:24:27,331:DEBUG::
2022-08-02 09:24:27,331:DEBUG::====================================================================================================
2022-08-02 09:24:27,331:INFO::Server Utils Section :: Logging Active
2022-08-02 09:24:27,331:DEBUG::
2022-08-03 09:17:33,590:DEBUG::
2022-08-03 09:17:33,602:DEBUG::====================================================================================================
2022-08-03 09:17:33,602:INFO::Server Utils Section :: Logging Active
2022-08-03 09:17:33,602:DEBUG::
2022-08-03 12:10:18,928:DEBUG::
2022-08-03 12:10:18,929:DEBUG::====================================================================================================
2022-08-03 12:10:18,929:INFO::Server Utils Section :: Logging Active
2022-08-03 12:10:18,929:DEBUG::
2022-08-03 12:56:23,107:DEBUG::
2022-08-03 12:56:23,107:DEBUG::====================================================================================================
2022-08-03 12:56:23,108:INFO::Server Utils Section :: Logging Active
2022-08-03 12:56:23,110:DEBUG::
2022-08-03 12:57:46,339:DEBUG::
2022-08-03 12:57:46,339:DEBUG::====================================================================================================
2022-08-03 12:57:46,339:INFO::Server Utils Section :: Logging Active
2022-08-03 12:57:46,339:DEBUG::
2022-08-03 12:57:56,441:DEBUG::
2022-08-03 12:57:56,441:DEBUG::====================================================================================================
2022-08-03 12:57:56,442:INFO::Server Utils Section :: Logging Active
2022-08-03 12:57:56,444:DEBUG::
2022-08-03 12:58:18,326:DEBUG::
2022-08-03 12:58:18,338:DEBUG::====================================================================================================
2022-08-03 12:58:18,338:INFO::Server Utils Section :: Logging Active
2022-08-03 12:58:18,338:DEBUG::
2022-08-03 14:16:56,549:DEBUG::
2022-08-03 14:16:56,561:DEBUG::====================================================================================================
2022-08-03 14:16:56,561:INFO::Server Utils Section :: Logging Active
2022-08-03 14:16:56,562:DEBUG::
2022-08-04 09:20:53,639:DEBUG::
2022-08-04 09:20:53,648:DEBUG::====================================================================================================
2022-08-04 09:20:53,648:INFO::Server Utils Section :: Logging Active
2022-08-04 09:20:53,648:DEBUG::
2022-08-04 13:00:10,347:DEBUG::
2022-08-04 13:00:10,347:DEBUG::====================================================================================================
2022-08-04 13:00:10,348:INFO::Server Utils Section :: Logging Active
2022-08-04 13:00:10,350:DEBUG::
2022-08-05 11:08:07,642:DEBUG::
2022-08-05 11:08:07,653:DEBUG::====================================================================================================
2022-08-05 11:08:07,653:INFO::Server Utils Section :: Logging Active
2022-08-05 11:08:07,653:DEBUG::
2022-08-08 11:08:03,711:DEBUG::
2022-08-08 11:08:03,747:DEBUG::====================================================================================================
2022-08-08 11:08:03,747:INFO::Server Utils Section :: Logging Active
2022-08-08 11:08:03,747:DEBUG::
2022-08-08 11:13:54,413:DEBUG::
2022-08-08 11:13:54,413:DEBUG::====================================================================================================
2022-08-08 11:13:54,414:INFO::Server Utils Section :: Logging Active
2022-08-08 11:13:54,414:DEBUG::
2022-08-08 11:33:46,605:DEBUG::
2022-08-08 11:33:46,605:DEBUG::====================================================================================================
2022-08-08 11:33:46,606:INFO::Server Utils Section :: Logging Active
2022-08-08 11:33:46,608:DEBUG::
2022-08-08 11:52:38,021:DEBUG::
2022-08-08 11:52:38,022:DEBUG::====================================================================================================
2022-08-08 11:52:38,022:INFO::Server Utils Section :: Logging Active
2022-08-08 11:52:38,022:DEBUG::
2022-08-08 11:53:50,590:DEBUG::FetchAdminError: Failed to Fetch Customers,'Admin' object has no attribute 'customer_push_notification_token'
2022-08-08 11:53:50,845:DEBUG::FetchAdminError: Failed to Fetch Customers,'Admin' object has no attribute 'customer_push_notification_token'
2022-08-08 11:54:05,048:DEBUG::FetchAdminError: Failed to Fetch Customers,'Admin' object has no attribute 'customer_push_notification_token'
2022-08-08 11:54:05,290:DEBUG::FetchAdminError: Failed to Fetch Customers,'Admin' object has no attribute 'customer_push_notification_token'
2022-08-08 11:56:32,444:DEBUG::
2022-08-08 11:56:32,444:DEBUG::====================================================================================================
2022-08-08 11:56:32,444:INFO::Server Utils Section :: Logging Active
2022-08-08 11:56:32,445:DEBUG::
2022-08-08 12:03:16,026:DEBUG::
2022-08-08 12:03:16,027:DEBUG::====================================================================================================
2022-08-08 12:03:16,028:INFO::Server Utils Section :: Logging Active
2022-08-08 12:03:16,030:DEBUG::
2022-08-08 12:05:31,346:DEBUG::
2022-08-08 12:05:31,347:DEBUG::====================================================================================================
2022-08-08 12:05:31,348:INFO::Server Utils Section :: Logging Active
2022-08-08 12:05:31,349:DEBUG::
2022-08-09 16:11:29,832:DEBUG::
2022-08-09 16:11:29,862:DEBUG::====================================================================================================
2022-08-09 16:11:29,862:INFO::Server Utils Section :: Logging Active
2022-08-09 16:11:29,862:DEBUG::
2022-08-10 09:17:25,851:DEBUG::
2022-08-10 09:17:25,858:DEBUG::====================================================================================================
2022-08-10 09:17:25,858:INFO::Server Utils Section :: Logging Active
2022-08-10 09:17:25,858:DEBUG::
2022-08-10 12:08:25,775:DEBUG::EditProductError: Failed to Edit Product,(builtins.TypeError) Not a boolean value: 'true'
[SQL: UPDATE products SET product_price=%s, product_discount=%s, product_is_available=%s WHERE products.id = %s]
[parameters: [{'product_price': '4998', 'product_discount': '10', 'product_is_available': 'true', 'products_id': 1}]]
2022-08-10 12:11:08,813:DEBUG::
2022-08-10 12:11:08,813:DEBUG::====================================================================================================
2022-08-10 12:11:08,814:INFO::Server Utils Section :: Logging Active
2022-08-10 12:11:08,816:DEBUG::
2022-08-10 12:12:23,224:DEBUG::EditProductError: Failed to Edit Product,(builtins.TypeError) Not a boolean value: '1'
[SQL: UPDATE products SET product_price=%s, product_discount=%s, product_is_available=%s WHERE products.id = %s]
[parameters: [{'product_price': '4998', 'product_is_available': '1', 'product_discount': '10', 'products_id': 1}]]
2022-08-10 12:16:40,191:DEBUG::EditProductError: Failed to Edit Product,(builtins.TypeError) Not a boolean value: 'True'
[SQL: UPDATE products SET product_price=%s, product_discount=%s, product_is_available=%s WHERE products.id = %s]
[parameters: [{'product_price': '4998', 'product_is_available': 'True', 'product_discount': '10', 'products_id': 1}]]
2022-08-10 12:18:08,405:DEBUG::
2022-08-10 12:18:08,405:DEBUG::====================================================================================================
2022-08-10 12:18:08,406:INFO::Server Utils Section :: Logging Active
2022-08-10 12:18:08,406:DEBUG::
2022-08-10 12:19:17,129:DEBUG::EditProductError: Failed to Edit Product,(builtins.TypeError) Not a boolean value: 'true'
[SQL: UPDATE products SET product_price=%s, product_discount=%s, product_is_available=%s WHERE products.id = %s]
[parameters: [{'product_is_available': 'true', 'product_discount': '10', 'product_price': '4991', 'products_id': 1}]]
2022-08-10 12:23:11,565:DEBUG::
2022-08-10 12:23:11,565:DEBUG::====================================================================================================
2022-08-10 12:23:11,565:INFO::Server Utils Section :: Logging Active
2022-08-10 12:23:11,565:DEBUG::
2022-08-10 12:23:41,534:DEBUG::EditProductError: Failed to Edit Product,(builtins.TypeError) Not a boolean value: 'true'
[SQL: UPDATE products SET product_price=%s, product_discount=%s, product_is_available=%s, product_category=%s WHERE products.id = %s]
[parameters: [{'product_discount': '10', 'product_is_available': 'true', 'product_price': '4998', 'product_category': False, 'products_id': 1}]]
2022-08-10 12:26:13,654:DEBUG::
2022-08-10 12:26:13,654:DEBUG::====================================================================================================
2022-08-10 12:26:13,655:INFO::Server Utils Section :: Logging Active
2022-08-10 12:26:13,655:DEBUG::
2022-08-10 14:11:18,144:DEBUG::
2022-08-10 14:11:18,145:DEBUG::====================================================================================================
2022-08-10 14:11:18,146:INFO::Server Utils Section :: Logging Active
2022-08-10 14:11:18,146:DEBUG::
2022-08-10 14:11:51,613:DEBUG::GetAllProductsError: Failed to Get All Products,'list' object has no attribute 'order_by'
2022-08-10 14:13:31,444:DEBUG::
2022-08-10 14:13:31,445:DEBUG::====================================================================================================
2022-08-10 14:13:31,445:INFO::Server Utils Section :: Logging Active
2022-08-10 14:13:31,445:DEBUG::
2022-08-10 14:13:58,206:DEBUG::
2022-08-10 14:13:58,206:DEBUG::====================================================================================================
2022-08-10 14:13:58,208:INFO::Server Utils Section :: Logging Active
2022-08-10 14:13:58,210:DEBUG::
2022-08-10 14:14:06,120:DEBUG::GetAllProductsError: Failed to Get All Products,Can't resolve label reference for ORDER BY / GROUP BY / DISTINCT etc. Textual SQL expression '-id' should be explicitly declared as text('-id')
2022-08-10 14:15:13,613:DEBUG::
2022-08-10 14:15:13,613:DEBUG::====================================================================================================
2022-08-10 14:15:13,614:INFO::Server Utils Section :: Logging Active
2022-08-10 14:15:13,614:DEBUG::
2022-08-10 14:15:48,296:DEBUG::
2022-08-10 14:15:48,297:DEBUG::====================================================================================================
2022-08-10 14:15:48,298:INFO::Server Utils Section :: Logging Active
2022-08-10 14:15:48,300:DEBUG::
2022-08-10 18:31:56,115:DEBUG::
2022-08-10 18:31:56,124:DEBUG::====================================================================================================
2022-08-10 18:31:56,124:INFO::Server Utils Section :: Logging Active
2022-08-10 18:31:56,124:DEBUG::
2022-08-10 18:34:47,717:DEBUG::
2022-08-10 18:34:47,717:DEBUG::====================================================================================================
2022-08-10 18:34:47,718:INFO::Server Utils Section :: Logging Active
2022-08-10 18:34:47,720:DEBUG::
2022-08-10 18:35:28,020:DEBUG::
2022-08-10 18:35:28,021:DEBUG::====================================================================================================
2022-08-10 18:35:28,021:INFO::Server Utils Section :: Logging Active
2022-08-10 18:35:28,021:DEBUG::
2022-08-10 18:37:16,156:DEBUG::
2022-08-10 18:37:16,156:DEBUG::====================================================================================================
2022-08-10 18:37:16,158:INFO::Server Utils Section :: Logging Active
2022-08-10 18:37:16,160:DEBUG::
2022-08-10 18:41:47,542:DEBUG::
2022-08-10 18:41:47,543:DEBUG::====================================================================================================
2022-08-10 18:41:47,543:INFO::Server Utils Section :: Logging Active
2022-08-10 18:41:47,543:DEBUG::
2022-08-10 18:53:46,683:DEBUG::
2022-08-10 18:53:46,683:DEBUG::====================================================================================================
2022-08-10 18:53:46,684:INFO::Server Utils Section :: Logging Active
2022-08-10 18:53:46,687:DEBUG::
2022-08-12 09:19:51,921:DEBUG::
2022-08-12 09:19:51,933:DEBUG::====================================================================================================
2022-08-12 09:19:51,933:INFO::Server Utils Section :: Logging Active
2022-08-12 09:19:51,933:DEBUG::
2022-08-12 12:27:08,645:DEBUG::
2022-08-12 12:27:08,646:DEBUG::====================================================================================================
2022-08-12 12:27:08,647:INFO::Server Utils Section :: Logging Active
2022-08-12 12:27:08,649:DEBUG::
2022-08-12 12:27:25,750:DEBUG::
2022-08-12 12:27:25,750:DEBUG::====================================================================================================
2022-08-12 12:27:25,750:INFO::Server Utils Section :: Logging Active
2022-08-12 12:27:25,751:DEBUG::
2022-08-12 12:47:08,637:DEBUG::
2022-08-12 12:47:08,637:DEBUG::====================================================================================================
2022-08-12 12:47:08,638:INFO::Server Utils Section :: Logging Active
2022-08-12 12:47:08,640:DEBUG::
2022-08-12 12:47:19,996:DEBUG::
2022-08-12 12:47:19,997:DEBUG::====================================================================================================
2022-08-12 12:47:19,998:INFO::Server Utils Section :: Logging Active
2022-08-12 12:47:19,999:DEBUG::
2022-08-12 12:47:24,211:DEBUG::
2022-08-12 12:47:24,212:DEBUG::====================================================================================================
2022-08-12 12:47:24,213:INFO::Server Utils Section :: Logging Active
2022-08-12 12:47:24,213:DEBUG::
2022-08-12 12:53:21,628:ERROR::AddPurchaseError: Failed to Add Purchase,'total_receipt_amount' is an invalid keyword argument for Receipts
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1591, in Add_Purchase
    receipt = Receipts(receipt_id = rec_id,total_receipt_amount=total_receipt_amount)
  File "<string>", line 4, in __init__
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\state.py", line 481, in _initialize_instance
    with util.safe_reraise():
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\compat.py", line 208, in raise_
    raise exception
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\state.py", line 479, in _initialize_instance
    return manager.original_init(*mixed[1:], **kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\decl_base.py", line 1163, in _declarative_constructor
    raise TypeError(
TypeError: 'total_receipt_amount' is an invalid keyword argument for Receipts
2022-08-12 12:54:43,402:DEBUG::
2022-08-12 12:54:43,402:DEBUG::====================================================================================================
2022-08-12 12:54:43,403:INFO::Server Utils Section :: Logging Active
2022-08-12 12:54:43,405:DEBUG::
2022-08-12 12:55:50,067:ERROR::AddPurchaseError: Failed to Add Purchase,(MySQLdb.IntegrityError) (1062, "Duplicate entry '2299' for key 'receipts.ix_receipts_receipt_total_amount'")
[SQL: INSERT INTO receipts (receipt_id, receipt_total_amount) VALUES (%s, %s)]
[parameters: ('ca19b10d6bb12487112bd7c088af13e7', 2299)]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1819, in _execute_context
    self.dialect.do_execute(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\default.py", line 732, in do_execute
    cursor.execute(statement, parameters)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\cursors.py", line 206, in execute
    res = self._query(query)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\cursors.py", line 319, in _query
    db.query(q)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\connections.py", line 254, in query
    _mysql.connection.query(self, query)
MySQLdb.IntegrityError: (1062, "Duplicate entry '2299' for key 'receipts.ix_receipts_receipt_total_amount'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1593, in Add_Purchase
    db.session.commit()
  File "<string>", line 2, in commit
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 1451, in commit
    self._transaction.commit(_to_root=self.future)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 829, in commit
    self._prepare_impl()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 808, in _prepare_impl
    self.session.flush()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 3383, in flush
    self._flush(objects)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 3522, in _flush
    with util.safe_reraise():
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\langhelpers.py", line 70, in __exit__
    compat.raise_(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\compat.py", line 208, in raise_
    raise exception
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 3483, in _flush
    flush_context.execute()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\unitofwork.py", line 456, in execute
    rec.execute(self)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\unitofwork.py", line 630, in execute
    util.preloaded.orm_persistence.save_obj(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\persistence.py", line 245, in save_obj
    _emit_insert_statements(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\persistence.py", line 1238, in _emit_insert_statements
    result = connection._execute_20(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1631, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\elements.py", line 332, in _execute_on_connection
    return connection._execute_clauseelement(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1498, in _execute_clauseelement
    ret = self._execute_context(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1862, in _execute_context
    self._handle_dbapi_exception(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 2043, in _handle_dbapi_exception
    util.raise_(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\compat.py", line 208, in raise_
    raise exception
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1819, in _execute_context
    self.dialect.do_execute(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\default.py", line 732, in do_execute
    cursor.execute(statement, parameters)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\cursors.py", line 206, in execute
    res = self._query(query)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\cursors.py", line 319, in _query
    db.query(q)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\MySQLdb\connections.py", line 254, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.IntegrityError: (MySQLdb.IntegrityError) (1062, "Duplicate entry '2299' for key 'receipts.ix_receipts_receipt_total_amount'")
[SQL: INSERT INTO receipts (receipt_id, receipt_total_amount) VALUES (%s, %s)]
[parameters: ('ca19b10d6bb12487112bd7c088af13e7', 2299)]
(Background on this error at: https://sqlalche.me/e/14/gkpj)
2022-08-12 12:56:34,874:DEBUG::
2022-08-12 12:56:34,875:DEBUG::====================================================================================================
2022-08-12 12:56:34,876:INFO::Server Utils Section :: Logging Active
2022-08-12 12:56:34,878:DEBUG::
2022-08-12 12:56:38,914:DEBUG::
2022-08-12 12:56:38,914:DEBUG::====================================================================================================
2022-08-12 12:56:38,916:INFO::Server Utils Section :: Logging Active
2022-08-12 12:56:38,918:DEBUG::
2022-08-12 12:56:44,425:DEBUG::
2022-08-12 12:56:44,425:DEBUG::====================================================================================================
2022-08-12 12:56:44,426:INFO::Server Utils Section :: Logging Active
2022-08-12 12:56:44,427:DEBUG::
2022-08-12 14:06:46,446:DEBUG::
2022-08-12 14:06:46,447:DEBUG::====================================================================================================
2022-08-12 14:06:46,448:INFO::Server Utils Section :: Logging Active
2022-08-12 14:06:46,448:DEBUG::
2022-08-16 09:20:22,375:DEBUG::
2022-08-16 09:20:22,383:DEBUG::====================================================================================================
2022-08-16 09:20:22,383:INFO::Server Utils Section :: Logging Active
2022-08-16 09:20:22,384:DEBUG::
2022-08-16 09:33:55,421:DEBUG::
2022-08-16 09:33:55,421:DEBUG::====================================================================================================
2022-08-16 09:33:55,422:INFO::Server Utils Section :: Logging Active
2022-08-16 09:33:55,424:DEBUG::
2022-08-16 09:34:04,917:DEBUG::
2022-08-16 09:34:04,917:DEBUG::====================================================================================================
2022-08-16 09:34:04,918:INFO::Server Utils Section :: Logging Active
2022-08-16 09:34:04,920:DEBUG::
2022-08-16 09:34:40,190:DEBUG::
2022-08-16 09:34:40,190:DEBUG::====================================================================================================
2022-08-16 09:34:40,191:INFO::Server Utils Section :: Logging Active
2022-08-16 09:34:40,193:DEBUG::
2022-08-16 09:37:04,468:DEBUG::
2022-08-16 09:37:04,468:DEBUG::====================================================================================================
2022-08-16 09:37:04,468:INFO::Server Utils Section :: Logging Active
2022-08-16 09:37:04,468:DEBUG::
2022-08-16 09:56:12,835:DEBUG::
2022-08-16 09:56:12,836:DEBUG::====================================================================================================
2022-08-16 09:56:12,837:INFO::Server Utils Section :: Logging Active
2022-08-16 09:56:12,839:DEBUG::
2022-08-16 09:56:16,829:DEBUG::
2022-08-16 09:56:16,830:DEBUG::====================================================================================================
2022-08-16 09:56:16,830:INFO::Server Utils Section :: Logging Active
2022-08-16 09:56:16,831:DEBUG::
2022-08-16 09:56:19,701:DEBUG::
2022-08-16 09:56:19,702:DEBUG::====================================================================================================
2022-08-16 09:56:19,703:INFO::Server Utils Section :: Logging Active
2022-08-16 09:56:19,705:DEBUG::
2022-08-16 10:07:38,878:DEBUG::
2022-08-16 10:07:38,879:DEBUG::====================================================================================================
2022-08-16 10:07:38,879:INFO::Server Utils Section :: Logging Active
2022-08-16 10:07:38,880:DEBUG::
2022-08-16 10:15:42,914:DEBUG::
2022-08-16 10:15:42,914:DEBUG::====================================================================================================
2022-08-16 10:15:42,916:INFO::Server Utils Section :: Logging Active
2022-08-16 10:15:42,917:DEBUG::
2022-08-16 10:24:46,951:DEBUG::
2022-08-16 10:24:46,951:DEBUG::====================================================================================================
2022-08-16 10:24:46,951:INFO::Server Utils Section :: Logging Active
2022-08-16 10:24:46,952:DEBUG::
2022-08-16 12:03:20,206:DEBUG::
2022-08-16 12:03:20,216:DEBUG::====================================================================================================
2022-08-16 12:03:20,218:INFO::Server Utils Section :: Logging Active
2022-08-16 12:03:20,218:DEBUG::
2022-08-16 12:03:28,398:ERROR::ShowPurchasesError: Failed to Show Purchases,'list' object has no attribute 'order_by'
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1683, in Show_Purchases
    receipts=Receipts.query.filter_by(customer_id=customer_id).all().order_by("receipt_date")
AttributeError: 'list' object has no attribute 'order_by'
2022-08-16 12:03:48,139:DEBUG::
2022-08-16 12:03:48,139:DEBUG::====================================================================================================
2022-08-16 12:03:48,140:INFO::Server Utils Section :: Logging Active
2022-08-16 12:03:48,141:DEBUG::
2022-08-16 12:04:03,580:DEBUG::
2022-08-16 12:04:03,580:DEBUG::====================================================================================================
2022-08-16 12:04:03,582:INFO::Server Utils Section :: Logging Active
2022-08-16 12:04:03,583:DEBUG::
2022-08-16 12:04:13,280:ERROR::ShowPurchasesError: Failed to Show Purchases,Can't resolve label reference for ORDER BY / GROUP BY / DISTINCT etc. Textual SQL expression '-receipt_date' should be explicitly declared as text('-receipt_date')
Traceback (most recent call last):
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 1452, in visit_textual_label_reference
    col = with_cols[element.element]
KeyError: '-receipt_date'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1683, in Show_Purchases
    receipts=Receipts.query.filter_by(customer_id=customer_id).order_by("-receipt_date").all()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\query.py", line 2768, in all
    return self._iter().all()
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\query.py", line 2903, in _iter
    result = self.session.execute(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\orm\session.py", line 1712, in execute
    result = conn._execute_20(statement, params or {}, execution_options)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1631, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\elements.py", line 332, in _execute_on_connection
    return connection._execute_clauseelement(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\engine\base.py", line 1490, in _execute_clauseelement
    compiled_sql, extracted_params, cache_hit = elem._compile_w_cache(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\elements.py", line 530, in _compile_w_cache
    compiled_sql = self._compiler(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\elements.py", line 565, in _compiler
    return dialect.statement_compiler(dialect, self, **kw)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 778, in __init__
    Compiled.__init__(self, dialect, statement, **kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 451, in __init__
    self.string = self.process(self.statement, **compile_kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 486, in process
    return obj._compiler_dispatch(self, **kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\visitors.py", line 82, in _compiler_dispatch
    return meth(self, **kw)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 3420, in visit_select
    text = self._compose_select_body(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 3598, in _compose_select_body
    text += self.order_by_clause(select, **kwargs)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 3710, in order_by_clause
    order_by = self._generate_delimited_list(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 1693, in _generate_delimited_list
    return separator.join(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 1693, in <genexpr>
    return separator.join(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 1695, in <genexpr>
    for s in (c._compiler_dispatch(self, **kw) for c in elements)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\visitors.py", line 82, in _compiler_dispatch
    return meth(self, **kw)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\compiler.py", line 1454, in visit_textual_label_reference
    coercions._no_text_coercion(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\sql\coercions.py", line 348, in _no_text_coercion
    util.raise_(
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python310\lib\site-packages\sqlalchemy\util\compat.py", line 208, in raise_
    raise exception
sqlalchemy.exc.CompileError: Can't resolve label reference for ORDER BY / GROUP BY / DISTINCT etc. Textual SQL expression '-receipt_date' should be explicitly declared as text('-receipt_date')
2022-08-16 12:06:19,245:DEBUG::
2022-08-16 12:06:19,246:DEBUG::====================================================================================================
2022-08-16 12:06:19,261:INFO::Server Utils Section :: Logging Active
2022-08-16 12:06:19,261:DEBUG::
2022-08-16 12:09:58,629:DEBUG::
2022-08-16 12:09:58,629:DEBUG::====================================================================================================
2022-08-16 12:09:58,630:INFO::Server Utils Section :: Logging Active
2022-08-16 12:09:58,632:DEBUG::
2022-08-16 12:10:05,356:DEBUG::
2022-08-16 12:10:05,356:DEBUG::====================================================================================================
2022-08-16 12:10:05,358:INFO::Server Utils Section :: Logging Active
2022-08-16 12:10:05,360:DEBUG::
2022-08-16 12:10:12,252:DEBUG::
2022-08-16 12:10:12,253:DEBUG::====================================================================================================
2022-08-16 12:10:12,253:INFO::Server Utils Section :: Logging Active
2022-08-16 12:10:12,254:DEBUG::
2022-08-16 12:36:00,146:DEBUG::
2022-08-16 12:36:00,147:DEBUG::====================================================================================================
2022-08-16 12:36:00,148:INFO::Server Utils Section :: Logging Active
2022-08-16 12:36:00,150:DEBUG::
2022-08-16 12:36:19,505:DEBUG::
2022-08-16 12:36:19,505:DEBUG::====================================================================================================
2022-08-16 12:36:19,506:INFO::Server Utils Section :: Logging Active
2022-08-16 12:36:19,509:DEBUG::
2022-08-16 12:38:15,866:DEBUG::
2022-08-16 12:38:15,867:DEBUG::====================================================================================================
2022-08-16 12:38:15,868:INFO::Server Utils Section :: Logging Active
2022-08-16 12:38:15,870:DEBUG::
2022-08-16 12:49:14,338:DEBUG::
2022-08-16 12:49:14,338:DEBUG::====================================================================================================
2022-08-16 12:49:14,339:INFO::Server Utils Section :: Logging Active
2022-08-16 12:49:14,341:DEBUG::
2022-08-16 14:18:50,129:DEBUG::
2022-08-16 14:18:50,139:DEBUG::====================================================================================================
2022-08-16 14:18:50,155:INFO::Server Utils Section :: Logging Active
2022-08-16 14:18:50,155:DEBUG::
2022-08-16 14:20:43,602:DEBUG::
2022-08-16 14:20:43,603:DEBUG::====================================================================================================
2022-08-16 14:20:43,604:INFO::Server Utils Section :: Logging Active
2022-08-16 14:20:43,606:DEBUG::
2022-08-16 14:38:11,963:DEBUG::
2022-08-16 14:38:11,963:DEBUG::====================================================================================================
2022-08-16 14:38:11,964:INFO::Server Utils Section :: Logging Active
2022-08-16 14:38:11,966:DEBUG::
2022-08-17 09:19:36,762:DEBUG::
2022-08-17 09:19:36,771:DEBUG::====================================================================================================
2022-08-17 09:19:36,771:INFO::Server Utils Section :: Logging Active
2022-08-17 09:19:36,772:DEBUG::
2022-08-17 10:30:19,098:DEBUG::
2022-08-17 10:30:19,099:DEBUG::====================================================================================================
2022-08-17 10:30:19,100:INFO::Server Utils Section :: Logging Active
2022-08-17 10:30:19,102:DEBUG::
2022-08-17 10:30:28,266:DEBUG::
2022-08-17 10:30:28,266:DEBUG::====================================================================================================
2022-08-17 10:30:28,267:INFO::Server Utils Section :: Logging Active
2022-08-17 10:30:28,269:DEBUG::
2022-08-17 10:30:44,646:DEBUG::
2022-08-17 10:30:44,647:DEBUG::====================================================================================================
2022-08-17 10:30:44,648:INFO::Server Utils Section :: Logging Active
2022-08-17 10:30:44,648:DEBUG::
2022-08-17 10:48:16,918:DEBUG::
2022-08-17 10:48:16,918:DEBUG::====================================================================================================
2022-08-17 10:48:16,919:INFO::Server Utils Section :: Logging Active
2022-08-17 10:48:16,919:DEBUG::
2022-08-17 10:50:12,238:DEBUG::
2022-08-17 10:50:12,238:DEBUG::====================================================================================================
2022-08-17 10:50:12,240:INFO::Server Utils Section :: Logging Active
2022-08-17 10:50:12,241:DEBUG::
2022-08-17 11:00:45,288:DEBUG::
2022-08-17 11:00:45,289:DEBUG::====================================================================================================
2022-08-17 11:00:45,297:INFO::Server Utils Section :: Logging Active
2022-08-17 11:00:45,297:DEBUG::
2022-08-17 11:14:18,878:DEBUG::
2022-08-17 11:14:18,879:DEBUG::====================================================================================================
2022-08-17 11:14:18,881:INFO::Server Utils Section :: Logging Active
2022-08-17 11:14:18,881:DEBUG::
2022-08-17 11:17:41,033:DEBUG::
2022-08-17 11:17:41,034:DEBUG::====================================================================================================
2022-08-17 11:17:41,035:INFO::Server Utils Section :: Logging Active
2022-08-17 11:17:41,036:DEBUG::
2022-08-17 11:19:17,509:DEBUG::
2022-08-17 11:19:17,509:DEBUG::====================================================================================================
2022-08-17 11:19:17,513:INFO::Server Utils Section :: Logging Active
2022-08-17 11:19:17,513:DEBUG::
2022-08-17 11:20:02,891:DEBUG::
2022-08-17 11:20:02,892:DEBUG::====================================================================================================
2022-08-17 11:20:02,893:INFO::Server Utils Section :: Logging Active
2022-08-17 11:20:02,893:DEBUG::
2022-08-17 11:20:10,324:ERROR::ShowPurchaseDetailsError: Failed to Show Purchase details,'list' object has no attribute 'receipt_id'
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1790, in Show_Purchases_by_id
    order_details=[{"receipt_id":order.receipt_id,"customer_id":order.order_customer_id,"date":order.order_date,"id":order.order_id,"product":[{"id":product.product_id,"name":product.product_name,"product_image":f"localhost:5003/static/images/products/{product.product_image_name}"} for product in Product.query.filter_by(product_id=order.order_product_id)],"quantity":order.order_product_quantity,"total_amount":order.order_total_amount,"address":order.order_address,"name":order.order_name,"phone":order.order_contact_no}]
AttributeError: 'list' object has no attribute 'receipt_id'
2022-08-17 11:21:57,142:DEBUG::
2022-08-17 11:21:57,142:DEBUG::====================================================================================================
2022-08-17 11:21:57,144:INFO::Server Utils Section :: Logging Active
2022-08-17 11:21:57,146:DEBUG::
2022-08-17 11:22:00,962:ERROR::ShowPurchaseDetailsError: Failed to Show Purchase details,'list' object has no attribute 'receipt_id'
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1791, in Show_Purchases_by_id
    order_details=[{"receipt_id":order.receipt_id,"customer_id":order.order_customer_id,"date":order.order_date,"id":order.order_id,"product":[{"id":product.product_id,"name":product.product_name,"product_image":f"localhost:5003/static/images/products/{product.product_image_name}"} for product in Product.query.filter_by(product_id=order.order_product_id)],"quantity":order.order_product_quantity,"total_amount":order.order_total_amount,"address":order.order_address,"name":order.order_name,"phone":order.order_contact_no}]
AttributeError: 'list' object has no attribute 'receipt_id'
2022-08-17 11:23:01,416:DEBUG::
2022-08-17 11:23:01,417:DEBUG::====================================================================================================
2022-08-17 11:23:01,418:INFO::Server Utils Section :: Logging Active
2022-08-17 11:23:01,419:DEBUG::
2022-08-17 11:27:58,957:DEBUG::
2022-08-17 11:27:58,959:DEBUG::====================================================================================================
2022-08-17 11:27:58,960:INFO::Server Utils Section :: Logging Active
2022-08-17 11:27:58,960:DEBUG::
2022-08-17 11:28:44,983:DEBUG::
2022-08-17 11:28:44,983:DEBUG::====================================================================================================
2022-08-17 11:28:44,984:INFO::Server Utils Section :: Logging Active
2022-08-17 11:28:44,988:DEBUG::
2022-08-18 09:15:43,561:DEBUG::
2022-08-18 09:15:43,571:DEBUG::====================================================================================================
2022-08-18 09:15:43,571:INFO::Server Utils Section :: Logging Active
2022-08-18 09:15:43,571:DEBUG::
2022-08-18 09:46:05,676:DEBUG::
2022-08-18 09:46:05,676:DEBUG::====================================================================================================
2022-08-18 09:46:05,677:INFO::Server Utils Section :: Logging Active
2022-08-18 09:46:05,680:DEBUG::
2022-08-18 09:47:01,742:DEBUG::
2022-08-18 09:47:01,742:DEBUG::====================================================================================================
2022-08-18 09:47:01,743:INFO::Server Utils Section :: Logging Active
2022-08-18 09:47:01,745:DEBUG::
2022-08-18 09:47:10,897:ERROR::ShowAllPurchasesError: Failed to Show all Purchases,'Customer' object is not iterable
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1731, in Show_all_Purchases
    orders_list = [{"order_id":order.order_id,"receipt_id":order.receipt_id,"order_total_amount":order.order_total_amount,"order_product_id":order.order_product_id,"order_status":order.order_status,"order_date":order.order_date,"customer":[{"id":customer.customer_id,"name":customer.customer_name} for customer in Customer.query.filter_by(customer_id=order.order_customer_id).first()]} for order in orders]
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1731, in <listcomp>
    orders_list = [{"order_id":order.order_id,"receipt_id":order.receipt_id,"order_total_amount":order.order_total_amount,"order_product_id":order.order_product_id,"order_status":order.order_status,"order_date":order.order_date,"customer":[{"id":customer.customer_id,"name":customer.customer_name} for customer in Customer.query.filter_by(customer_id=order.order_customer_id).first()]} for order in orders]
TypeError: 'Customer' object is not iterable
2022-08-18 09:48:54,864:DEBUG::
2022-08-18 09:48:54,864:DEBUG::====================================================================================================
2022-08-18 09:48:54,865:INFO::Server Utils Section :: Logging Active
2022-08-18 09:48:54,867:DEBUG::
2022-08-18 09:49:06,453:ERROR::ShowAllPurchasesError: Failed to Show all Purchases,'method' object is not iterable
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1731, in Show_all_Purchases
    orders_list = [{"order_id":order.order_id,"receipt_id":order.receipt_id,"order_total_amount":order.order_total_amount,"order_product_id":order.order_product_id,"order_status":order.order_status,"order_date":order.order_date,"customer":[{"id":customer.customer_id,"name":customer.customer_name} for customer in Customer.query.filter_by(customer_id=order.order_customer_id).first][0]} for order in orders]
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1731, in <listcomp>
    orders_list = [{"order_id":order.order_id,"receipt_id":order.receipt_id,"order_total_amount":order.order_total_amount,"order_product_id":order.order_product_id,"order_status":order.order_status,"order_date":order.order_date,"customer":[{"id":customer.customer_id,"name":customer.customer_name} for customer in Customer.query.filter_by(customer_id=order.order_customer_id).first][0]} for order in orders]
TypeError: 'method' object is not iterable
2022-08-18 09:49:52,551:DEBUG::
2022-08-18 09:49:52,552:DEBUG::====================================================================================================
2022-08-18 09:49:52,553:INFO::Server Utils Section :: Logging Active
2022-08-18 09:49:52,555:DEBUG::
2022-08-18 10:05:07,957:DEBUG::
2022-08-18 10:05:07,957:DEBUG::====================================================================================================
2022-08-18 10:05:07,958:INFO::Server Utils Section :: Logging Active
2022-08-18 10:05:07,959:DEBUG::
2022-08-18 10:40:58,478:DEBUG::
2022-08-18 10:40:58,478:DEBUG::====================================================================================================
2022-08-18 10:40:58,480:INFO::Server Utils Section :: Logging Active
2022-08-18 10:40:58,482:DEBUG::
2022-08-18 18:09:07,601:DEBUG::
2022-08-18 18:09:07,630:DEBUG::====================================================================================================
2022-08-18 18:09:07,631:INFO::Server Utils Section :: Logging Active
2022-08-18 18:09:07,631:DEBUG::
2022-08-18 18:23:10,434:DEBUG::
2022-08-18 18:23:10,434:DEBUG::====================================================================================================
2022-08-18 18:23:10,434:INFO::Server Utils Section :: Logging Active
2022-08-18 18:23:10,435:DEBUG::
2022-08-18 18:23:52,417:DEBUG::
2022-08-18 18:23:52,417:DEBUG::====================================================================================================
2022-08-18 18:23:52,418:INFO::Server Utils Section :: Logging Active
2022-08-18 18:23:52,420:DEBUG::
2022-08-18 18:23:57,020:DEBUG::EditProductError: Failed to Edit Product,type object 'Orders' has no attribute 'objects'
2022-08-18 18:25:06,863:DEBUG::
2022-08-18 18:25:06,863:DEBUG::====================================================================================================
2022-08-18 18:25:06,863:INFO::Server Utils Section :: Logging Active
2022-08-18 18:25:06,864:DEBUG::
2022-08-19 09:15:23,845:DEBUG::
2022-08-19 09:15:23,852:DEBUG::====================================================================================================
2022-08-19 09:15:23,852:INFO::Server Utils Section :: Logging Active
2022-08-19 09:15:23,852:DEBUG::
2022-08-19 09:43:24,071:DEBUG::
2022-08-19 09:43:24,071:DEBUG::====================================================================================================
2022-08-19 09:43:24,072:INFO::Server Utils Section :: Logging Active
2022-08-19 09:43:24,074:DEBUG::
2022-08-19 10:23:48,264:ERROR::ShowPurchaseDetailsError: Failed to Show Purchase details,'NoneType' object has no attribute 'receipt_id'
Traceback (most recent call last):
  File "D:\React\PL\grand_sprorts\backend-main\backend-main\app\server_utils.py", line 1793, in Show_Purchases_by_id
    order_details=[{"receipt_id":order.receipt_id,"customer_id":order.order_customer_id,"order_status":order.order_status,"order_tracking_id":order.order_tracking_id,"order_delivery_partner":order.order_delivery_partner,"date":order.order_date,"id":order.order_id,"product":[{"id":product.product_id,"name":product.product_name,"product_image":f"localhost:5003/static/images/products/{product.product_image_name}"} for product in Product.query.filter_by(product_id=order.order_product_id)][0],"quantity":order.order_product_quantity,"total_amount":order.order_total_amount,"address":order.order_address,"name":order.order_name,"phone":order.order_contact_no}]
AttributeError: 'NoneType' object has no attribute 'receipt_id'
